// @aws-sdk/client-ec2/dist-cjs/models/models_5.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.DetachInternetGatewayRequestFilterSensitiveLog = exports.DetachClassicLinkVpcResultFilterSensitiveLog = exports.DetachClassicLinkVpcRequestFilterSensitiveLog = exports.DescribeVpnGatewaysResultFilterSensitiveLog = exports.DescribeVpnGatewaysRequestFilterSensitiveLog = exports.DescribeVpnConnectionsResultFilterSensitiveLog = exports.DescribeVpnConnectionsRequestFilterSensitiveLog = exports.DescribeVpcsResultFilterSensitiveLog = exports.DescribeVpcsRequestFilterSensitiveLog = exports.DescribeVpcPeeringConnectionsResultFilterSensitiveLog = exports.DescribeVpcPeeringConnectionsRequestFilterSensitiveLog = exports.DescribeVpcEndpointServicesResultFilterSensitiveLog = exports.ServiceDetailFilterSensitiveLog = exports.PrivateDnsDetailsFilterSensitiveLog = exports.DescribeVpcEndpointServicesRequestFilterSensitiveLog = exports.DescribeVpcEndpointServicePermissionsResultFilterSensitiveLog = exports.DescribeVpcEndpointServicePermissionsRequestFilterSensitiveLog = exports.DescribeVpcEndpointServiceConfigurationsResultFilterSensitiveLog = exports.DescribeVpcEndpointServiceConfigurationsRequestFilterSensitiveLog = exports.DescribeVpcEndpointsResultFilterSensitiveLog = exports.DescribeVpcEndpointsRequestFilterSensitiveLog = exports.DescribeVpcEndpointConnectionsResultFilterSensitiveLog = exports.VpcEndpointConnectionFilterSensitiveLog = exports.DescribeVpcEndpointConnectionsRequestFilterSensitiveLog = exports.DescribeVpcEndpointConnectionNotificationsResultFilterSensitiveLog = exports.DescribeVpcEndpointConnectionNotificationsRequestFilterSensitiveLog = exports.DescribeVpcClassicLinkDnsSupportResultFilterSensitiveLog = exports.ClassicLinkDnsSupportFilterSensitiveLog = exports.DescribeVpcClassicLinkDnsSupportRequestFilterSensitiveLog = exports.DescribeVpcClassicLinkResultFilterSensitiveLog = exports.VpcClassicLinkFilterSensitiveLog = exports.DescribeVpcClassicLinkRequestFilterSensitiveLog = exports.DescribeVpcAttributeResultFilterSensitiveLog = exports.DescribeVpcAttributeRequestFilterSensitiveLog = exports.DescribeVolumeStatusResultFilterSensitiveLog = exports.VolumeStatusItemFilterSensitiveLog = exports.VolumeStatusInfoFilterSensitiveLog = exports.VolumeStatusDetailsFilterSensitiveLog = exports.IpamManagementState = exports.IpamResourceType = exports.IpamDiscoveryFailureCode = exports.IpamAddressHistoryResourceType = exports.IpamOverlapStatus = exports.IpamComplianceStatus = exports.PartitionLoadFrequency = exports.UnlimitedSupportedInstanceFamily = exports.ClientCertificateRevocationListStatusCode = exports.TransitGatewayPropagationState = exports.VpcAttributeName = exports.VolumeStatusInfoStatus = void 0;
exports.DisassociateSubnetCidrBlockResultFilterSensitiveLog = exports.DisassociateSubnetCidrBlockRequestFilterSensitiveLog = exports.DisassociateRouteTableRequestFilterSensitiveLog = exports.DisassociateNatGatewayAddressResultFilterSensitiveLog = exports.DisassociateNatGatewayAddressRequestFilterSensitiveLog = exports.DisassociateIpamResourceDiscoveryResultFilterSensitiveLog = exports.DisassociateIpamResourceDiscoveryRequestFilterSensitiveLog = exports.DisassociateInstanceEventWindowResultFilterSensitiveLog = exports.DisassociateInstanceEventWindowRequestFilterSensitiveLog = exports.InstanceEventWindowDisassociationRequestFilterSensitiveLog = exports.DisassociateIamInstanceProfileResultFilterSensitiveLog = exports.DisassociateIamInstanceProfileRequestFilterSensitiveLog = exports.DisassociateEnclaveCertificateIamRoleResultFilterSensitiveLog = exports.DisassociateEnclaveCertificateIamRoleRequestFilterSensitiveLog = exports.DisassociateClientVpnTargetNetworkResultFilterSensitiveLog = exports.DisassociateClientVpnTargetNetworkRequestFilterSensitiveLog = exports.DisassociateAddressRequestFilterSensitiveLog = exports.DisableVpcClassicLinkDnsSupportResultFilterSensitiveLog = exports.DisableVpcClassicLinkDnsSupportRequestFilterSensitiveLog = exports.DisableVpcClassicLinkResultFilterSensitiveLog = exports.DisableVpcClassicLinkRequestFilterSensitiveLog = exports.DisableVgwRoutePropagationRequestFilterSensitiveLog = exports.DisableTransitGatewayRouteTablePropagationResultFilterSensitiveLog = exports.TransitGatewayPropagationFilterSensitiveLog = exports.DisableTransitGatewayRouteTablePropagationRequestFilterSensitiveLog = exports.DisableSerialConsoleAccessResultFilterSensitiveLog = exports.DisableSerialConsoleAccessRequestFilterSensitiveLog = exports.DisableIpamOrganizationAdminAccountResultFilterSensitiveLog = exports.DisableIpamOrganizationAdminAccountRequestFilterSensitiveLog = exports.DisableImageDeprecationResultFilterSensitiveLog = exports.DisableImageDeprecationRequestFilterSensitiveLog = exports.DisableFastSnapshotRestoresResultFilterSensitiveLog = exports.DisableFastSnapshotRestoreErrorItemFilterSensitiveLog = exports.DisableFastSnapshotRestoreStateErrorItemFilterSensitiveLog = exports.DisableFastSnapshotRestoreStateErrorFilterSensitiveLog = exports.DisableFastSnapshotRestoreSuccessItemFilterSensitiveLog = exports.DisableFastSnapshotRestoresRequestFilterSensitiveLog = exports.DisableFastLaunchResultFilterSensitiveLog = exports.DisableFastLaunchRequestFilterSensitiveLog = exports.DisableEbsEncryptionByDefaultResultFilterSensitiveLog = exports.DisableEbsEncryptionByDefaultRequestFilterSensitiveLog = exports.DisableAwsNetworkPerformanceMetricSubscriptionResultFilterSensitiveLog = exports.DisableAwsNetworkPerformanceMetricSubscriptionRequestFilterSensitiveLog = exports.DisableAddressTransferResultFilterSensitiveLog = exports.DisableAddressTransferRequestFilterSensitiveLog = exports.DetachVpnGatewayRequestFilterSensitiveLog = exports.DetachVolumeRequestFilterSensitiveLog = exports.DetachVerifiedAccessTrustProviderResultFilterSensitiveLog = exports.DetachVerifiedAccessTrustProviderRequestFilterSensitiveLog = exports.DetachNetworkInterfaceRequestFilterSensitiveLog = void 0;
exports.ExportImageResultFilterSensitiveLog = exports.ExportImageRequestFilterSensitiveLog = exports.ExportTaskS3LocationRequestFilterSensitiveLog = exports.ExportClientVpnClientConfigurationResultFilterSensitiveLog = exports.ExportClientVpnClientConfigurationRequestFilterSensitiveLog = exports.ExportClientVpnClientCertificateRevocationListResultFilterSensitiveLog = exports.ClientCertificateRevocationListStatusFilterSensitiveLog = exports.ExportClientVpnClientCertificateRevocationListRequestFilterSensitiveLog = exports.EnableVpcClassicLinkDnsSupportResultFilterSensitiveLog = exports.EnableVpcClassicLinkDnsSupportRequestFilterSensitiveLog = exports.EnableVpcClassicLinkResultFilterSensitiveLog = exports.EnableVpcClassicLinkRequestFilterSensitiveLog = exports.EnableVolumeIORequestFilterSensitiveLog = exports.EnableVgwRoutePropagationRequestFilterSensitiveLog = exports.EnableTransitGatewayRouteTablePropagationResultFilterSensitiveLog = exports.EnableTransitGatewayRouteTablePropagationRequestFilterSensitiveLog = exports.EnableSerialConsoleAccessResultFilterSensitiveLog = exports.EnableSerialConsoleAccessRequestFilterSensitiveLog = exports.EnableReachabilityAnalyzerOrganizationSharingResultFilterSensitiveLog = exports.EnableReachabilityAnalyzerOrganizationSharingRequestFilterSensitiveLog = exports.EnableIpamOrganizationAdminAccountResultFilterSensitiveLog = exports.EnableIpamOrganizationAdminAccountRequestFilterSensitiveLog = exports.EnableImageDeprecationResultFilterSensitiveLog = exports.EnableImageDeprecationRequestFilterSensitiveLog = exports.EnableFastSnapshotRestoresResultFilterSensitiveLog = exports.EnableFastSnapshotRestoreErrorItemFilterSensitiveLog = exports.EnableFastSnapshotRestoreStateErrorItemFilterSensitiveLog = exports.EnableFastSnapshotRestoreStateErrorFilterSensitiveLog = exports.EnableFastSnapshotRestoreSuccessItemFilterSensitiveLog = exports.EnableFastSnapshotRestoresRequestFilterSensitiveLog = exports.EnableFastLaunchResultFilterSensitiveLog = exports.EnableFastLaunchRequestFilterSensitiveLog = exports.FastLaunchSnapshotConfigurationRequestFilterSensitiveLog = exports.FastLaunchLaunchTemplateSpecificationRequestFilterSensitiveLog = exports.EnableEbsEncryptionByDefaultResultFilterSensitiveLog = exports.EnableEbsEncryptionByDefaultRequestFilterSensitiveLog = exports.EnableAwsNetworkPerformanceMetricSubscriptionResultFilterSensitiveLog = exports.EnableAwsNetworkPerformanceMetricSubscriptionRequestFilterSensitiveLog = exports.EnableAddressTransferResultFilterSensitiveLog = exports.EnableAddressTransferRequestFilterSensitiveLog = exports.DisassociateVpcCidrBlockResultFilterSensitiveLog = exports.DisassociateVpcCidrBlockRequestFilterSensitiveLog = exports.DisassociateTrunkInterfaceResultFilterSensitiveLog = exports.DisassociateTrunkInterfaceRequestFilterSensitiveLog = exports.DisassociateTransitGatewayRouteTableResultFilterSensitiveLog = exports.DisassociateTransitGatewayRouteTableRequestFilterSensitiveLog = exports.DisassociateTransitGatewayPolicyTableResultFilterSensitiveLog = exports.DisassociateTransitGatewayPolicyTableRequestFilterSensitiveLog = exports.DisassociateTransitGatewayMulticastDomainResultFilterSensitiveLog = exports.DisassociateTransitGatewayMulticastDomainRequestFilterSensitiveLog = void 0;
exports.IpamDiscoveryFailureReasonFilterSensitiveLog = exports.GetIpamDiscoveredAccountsRequestFilterSensitiveLog = exports.GetIpamAddressHistoryResultFilterSensitiveLog = exports.IpamAddressHistoryRecordFilterSensitiveLog = exports.GetIpamAddressHistoryRequestFilterSensitiveLog = exports.GetInstanceUefiDataResultFilterSensitiveLog = exports.GetInstanceUefiDataRequestFilterSensitiveLog = exports.GetInstanceTypesFromInstanceRequirementsResultFilterSensitiveLog = exports.InstanceTypeInfoFromInstanceRequirementsFilterSensitiveLog = exports.GetInstanceTypesFromInstanceRequirementsRequestFilterSensitiveLog = exports.GetHostReservationPurchasePreviewResultFilterSensitiveLog = exports.PurchaseFilterSensitiveLog = exports.GetHostReservationPurchasePreviewRequestFilterSensitiveLog = exports.GetGroupsForCapacityReservationResultFilterSensitiveLog = exports.CapacityReservationGroupFilterSensitiveLog = exports.GetGroupsForCapacityReservationRequestFilterSensitiveLog = exports.GetFlowLogsIntegrationTemplateResultFilterSensitiveLog = exports.GetFlowLogsIntegrationTemplateRequestFilterSensitiveLog = exports.IntegrateServicesFilterSensitiveLog = exports.AthenaIntegrationFilterSensitiveLog = exports.GetEbsEncryptionByDefaultResultFilterSensitiveLog = exports.GetEbsEncryptionByDefaultRequestFilterSensitiveLog = exports.GetEbsDefaultKmsKeyIdResultFilterSensitiveLog = exports.GetEbsDefaultKmsKeyIdRequestFilterSensitiveLog = exports.GetDefaultCreditSpecificationResultFilterSensitiveLog = exports.InstanceFamilyCreditSpecificationFilterSensitiveLog = exports.GetDefaultCreditSpecificationRequestFilterSensitiveLog = exports.GetConsoleScreenshotResultFilterSensitiveLog = exports.GetConsoleScreenshotRequestFilterSensitiveLog = exports.GetConsoleOutputResultFilterSensitiveLog = exports.GetConsoleOutputRequestFilterSensitiveLog = exports.GetCoipPoolUsageResultFilterSensitiveLog = exports.CoipAddressUsageFilterSensitiveLog = exports.GetCoipPoolUsageRequestFilterSensitiveLog = exports.GetCapacityReservationUsageResultFilterSensitiveLog = exports.InstanceUsageFilterSensitiveLog = exports.GetCapacityReservationUsageRequestFilterSensitiveLog = exports.GetAwsNetworkPerformanceDataResultFilterSensitiveLog = exports.DataResponseFilterSensitiveLog = exports.MetricPointFilterSensitiveLog = exports.GetAwsNetworkPerformanceDataRequestFilterSensitiveLog = exports.DataQueryFilterSensitiveLog = exports.GetAssociatedIpv6PoolCidrsResultFilterSensitiveLog = exports.Ipv6CidrAssociationFilterSensitiveLog = exports.GetAssociatedIpv6PoolCidrsRequestFilterSensitiveLog = exports.GetAssociatedEnclaveCertificateIamRolesResultFilterSensitiveLog = exports.AssociatedRoleFilterSensitiveLog = exports.GetAssociatedEnclaveCertificateIamRolesRequestFilterSensitiveLog = exports.ExportTransitGatewayRoutesResultFilterSensitiveLog = exports.ExportTransitGatewayRoutesRequestFilterSensitiveLog = void 0;
exports.TransitGatewayPolicyRuleMetaDataFilterSensitiveLog = exports.GetTransitGatewayPolicyTableEntriesRequestFilterSensitiveLog = exports.GetTransitGatewayPolicyTableAssociationsResultFilterSensitiveLog = exports.GetTransitGatewayPolicyTableAssociationsRequestFilterSensitiveLog = exports.GetTransitGatewayMulticastDomainAssociationsResultFilterSensitiveLog = exports.TransitGatewayMulticastDomainAssociationFilterSensitiveLog = exports.GetTransitGatewayMulticastDomainAssociationsRequestFilterSensitiveLog = exports.GetTransitGatewayAttachmentPropagationsResultFilterSensitiveLog = exports.TransitGatewayAttachmentPropagationFilterSensitiveLog = exports.GetTransitGatewayAttachmentPropagationsRequestFilterSensitiveLog = exports.GetSubnetCidrReservationsResultFilterSensitiveLog = exports.GetSubnetCidrReservationsRequestFilterSensitiveLog = exports.GetSpotPlacementScoresResultFilterSensitiveLog = exports.SpotPlacementScoreFilterSensitiveLog = exports.GetSpotPlacementScoresRequestFilterSensitiveLog = exports.InstanceRequirementsWithMetadataRequestFilterSensitiveLog = exports.GetSerialConsoleAccessStatusResultFilterSensitiveLog = exports.GetSerialConsoleAccessStatusRequestFilterSensitiveLog = exports.GetReservedInstancesExchangeQuoteResultFilterSensitiveLog = exports.TargetReservationValueFilterSensitiveLog = exports.TargetConfigurationFilterSensitiveLog = exports.ReservedInstanceReservationValueFilterSensitiveLog = exports.ReservationValueFilterSensitiveLog = exports.GetReservedInstancesExchangeQuoteRequestFilterSensitiveLog = exports.GetPasswordDataResultFilterSensitiveLog = exports.GetPasswordDataRequestFilterSensitiveLog = exports.GetNetworkInsightsAccessScopeContentResultFilterSensitiveLog = exports.GetNetworkInsightsAccessScopeContentRequestFilterSensitiveLog = exports.GetNetworkInsightsAccessScopeAnalysisFindingsResultFilterSensitiveLog = exports.GetNetworkInsightsAccessScopeAnalysisFindingsRequestFilterSensitiveLog = exports.GetManagedPrefixListEntriesResultFilterSensitiveLog = exports.PrefixListEntryFilterSensitiveLog = exports.GetManagedPrefixListEntriesRequestFilterSensitiveLog = exports.GetManagedPrefixListAssociationsResultFilterSensitiveLog = exports.PrefixListAssociationFilterSensitiveLog = exports.GetManagedPrefixListAssociationsRequestFilterSensitiveLog = exports.GetLaunchTemplateDataResultFilterSensitiveLog = exports.GetLaunchTemplateDataRequestFilterSensitiveLog = exports.GetIpamResourceCidrsResultFilterSensitiveLog = exports.IpamResourceCidrFilterSensitiveLog = exports.GetIpamResourceCidrsRequestFilterSensitiveLog = exports.GetIpamPoolCidrsResultFilterSensitiveLog = exports.GetIpamPoolCidrsRequestFilterSensitiveLog = exports.GetIpamPoolAllocationsResultFilterSensitiveLog = exports.GetIpamPoolAllocationsRequestFilterSensitiveLog = exports.GetIpamDiscoveredResourceCidrsResultFilterSensitiveLog = exports.IpamDiscoveredResourceCidrFilterSensitiveLog = exports.GetIpamDiscoveredResourceCidrsRequestFilterSensitiveLog = exports.GetIpamDiscoveredAccountsResultFilterSensitiveLog = exports.IpamDiscoveredAccountFilterSensitiveLog = void 0;
exports.ModifyAddressAttributeResultFilterSensitiveLog = exports.ModifyAddressAttributeRequestFilterSensitiveLog = exports.ListSnapshotsInRecycleBinResultFilterSensitiveLog = exports.SnapshotRecycleBinInfoFilterSensitiveLog = exports.ListSnapshotsInRecycleBinRequestFilterSensitiveLog = exports.ListImagesInRecycleBinResultFilterSensitiveLog = exports.ImageRecycleBinInfoFilterSensitiveLog = exports.ListImagesInRecycleBinRequestFilterSensitiveLog = exports.ImportVolumeResultFilterSensitiveLog = exports.ImportVolumeRequestFilterSensitiveLog = exports.ImportSnapshotResultFilterSensitiveLog = exports.ImportSnapshotRequestFilterSensitiveLog = exports.SnapshotDiskContainerFilterSensitiveLog = exports.ImportKeyPairResultFilterSensitiveLog = exports.ImportKeyPairRequestFilterSensitiveLog = exports.ImportInstanceResultFilterSensitiveLog = exports.ImportInstanceRequestFilterSensitiveLog = exports.ImportInstanceLaunchSpecificationFilterSensitiveLog = exports.UserDataFilterSensitiveLog = exports.DiskImageFilterSensitiveLog = exports.VolumeDetailFilterSensitiveLog = exports.DiskImageDetailFilterSensitiveLog = exports.ImportImageResultFilterSensitiveLog = exports.ImportImageRequestFilterSensitiveLog = exports.ImportImageLicenseConfigurationRequestFilterSensitiveLog = exports.ImageDiskContainerFilterSensitiveLog = exports.UserBucketFilterSensitiveLog = exports.ClientDataFilterSensitiveLog = exports.ImportClientVpnClientCertificateRevocationListResultFilterSensitiveLog = exports.ImportClientVpnClientCertificateRevocationListRequestFilterSensitiveLog = exports.GetVpnConnectionDeviceTypesResultFilterSensitiveLog = exports.VpnConnectionDeviceTypeFilterSensitiveLog = exports.GetVpnConnectionDeviceTypesRequestFilterSensitiveLog = exports.GetVpnConnectionDeviceSampleConfigurationResultFilterSensitiveLog = exports.GetVpnConnectionDeviceSampleConfigurationRequestFilterSensitiveLog = exports.GetVerifiedAccessGroupPolicyResultFilterSensitiveLog = exports.GetVerifiedAccessGroupPolicyRequestFilterSensitiveLog = exports.GetVerifiedAccessEndpointPolicyResultFilterSensitiveLog = exports.GetVerifiedAccessEndpointPolicyRequestFilterSensitiveLog = exports.GetTransitGatewayRouteTablePropagationsResultFilterSensitiveLog = exports.TransitGatewayRouteTablePropagationFilterSensitiveLog = exports.GetTransitGatewayRouteTablePropagationsRequestFilterSensitiveLog = exports.GetTransitGatewayRouteTableAssociationsResultFilterSensitiveLog = exports.TransitGatewayRouteTableAssociationFilterSensitiveLog = exports.GetTransitGatewayRouteTableAssociationsRequestFilterSensitiveLog = exports.GetTransitGatewayPrefixListReferencesResultFilterSensitiveLog = exports.GetTransitGatewayPrefixListReferencesRequestFilterSensitiveLog = exports.GetTransitGatewayPolicyTableEntriesResultFilterSensitiveLog = exports.TransitGatewayPolicyTableEntryFilterSensitiveLog = exports.TransitGatewayPolicyRuleFilterSensitiveLog = void 0;
const smithy_client_1 = require("@aws-sdk/smithy-client");
const models_1_1 = require("./models_1");
var VolumeStatusInfoStatus;
(function (VolumeStatusInfoStatus) {
    VolumeStatusInfoStatus["impaired"] = "impaired";
    VolumeStatusInfoStatus["insufficient_data"] = "insufficient-data";
    VolumeStatusInfoStatus["ok"] = "ok";
})(VolumeStatusInfoStatus = exports.VolumeStatusInfoStatus || (exports.VolumeStatusInfoStatus = {}));
var VpcAttributeName;
(function (VpcAttributeName) {
    VpcAttributeName["enableDnsHostnames"] = "enableDnsHostnames";
    VpcAttributeName["enableDnsSupport"] = "enableDnsSupport";
    VpcAttributeName["enableNetworkAddressUsageMetrics"] = "enableNetworkAddressUsageMetrics";
})(VpcAttributeName = exports.VpcAttributeName || (exports.VpcAttributeName = {}));
var TransitGatewayPropagationState;
(function (TransitGatewayPropagationState) {
    TransitGatewayPropagationState["disabled"] = "disabled";
    TransitGatewayPropagationState["disabling"] = "disabling";
    TransitGatewayPropagationState["enabled"] = "enabled";
    TransitGatewayPropagationState["enabling"] = "enabling";
})(TransitGatewayPropagationState = exports.TransitGatewayPropagationState || (exports.TransitGatewayPropagationState = {}));
var ClientCertificateRevocationListStatusCode;
(function (ClientCertificateRevocationListStatusCode) {
    ClientCertificateRevocationListStatusCode["active"] = "active";
    ClientCertificateRevocationListStatusCode["pending"] = "pending";
})(ClientCertificateRevocationListStatusCode = exports.ClientCertificateRevocationListStatusCode || (exports.ClientCertificateRevocationListStatusCode = {}));
var UnlimitedSupportedInstanceFamily;
(function (UnlimitedSupportedInstanceFamily) {
    UnlimitedSupportedInstanceFamily["t2"] = "t2";
    UnlimitedSupportedInstanceFamily["t3"] = "t3";
    UnlimitedSupportedInstanceFamily["t3a"] = "t3a";
    UnlimitedSupportedInstanceFamily["t4g"] = "t4g";
})(UnlimitedSupportedInstanceFamily = exports.UnlimitedSupportedInstanceFamily || (exports.UnlimitedSupportedInstanceFamily = {}));
var PartitionLoadFrequency;
(function (PartitionLoadFrequency) {
    PartitionLoadFrequency["DAILY"] = "daily";
    PartitionLoadFrequency["MONTHLY"] = "monthly";
    PartitionLoadFrequency["NONE"] = "none";
    PartitionLoadFrequency["WEEKLY"] = "weekly";
})(PartitionLoadFrequency = exports.PartitionLoadFrequency || (exports.PartitionLoadFrequency = {}));
var IpamComplianceStatus;
(function (IpamComplianceStatus) {
    IpamComplianceStatus["compliant"] = "compliant";
    IpamComplianceStatus["ignored"] = "ignored";
    IpamComplianceStatus["noncompliant"] = "noncompliant";
    IpamComplianceStatus["unmanaged"] = "unmanaged";
})(IpamComplianceStatus = exports.IpamComplianceStatus || (exports.IpamComplianceStatus = {}));
var IpamOverlapStatus;
(function (IpamOverlapStatus) {
    IpamOverlapStatus["ignored"] = "ignored";
    IpamOverlapStatus["nonoverlapping"] = "nonoverlapping";
    IpamOverlapStatus["overlapping"] = "overlapping";
})(IpamOverlapStatus = exports.IpamOverlapStatus || (exports.IpamOverlapStatus = {}));
var IpamAddressHistoryResourceType;
(function (IpamAddressHistoryResourceType) {
    IpamAddressHistoryResourceType["eip"] = "eip";
    IpamAddressHistoryResourceType["instance"] = "instance";
    IpamAddressHistoryResourceType["network_interface"] = "network-interface";
    IpamAddressHistoryResourceType["subnet"] = "subnet";
    IpamAddressHistoryResourceType["vpc"] = "vpc";
})(IpamAddressHistoryResourceType = exports.IpamAddressHistoryResourceType || (exports.IpamAddressHistoryResourceType = {}));
var IpamDiscoveryFailureCode;
(function (IpamDiscoveryFailureCode) {
    IpamDiscoveryFailureCode["assume_role_failure"] = "assume-role-failure";
    IpamDiscoveryFailureCode["throttling_failure"] = "throttling-failure";
    IpamDiscoveryFailureCode["unauthorized_failure"] = "unauthorized-failure";
})(IpamDiscoveryFailureCode = exports.IpamDiscoveryFailureCode || (exports.IpamDiscoveryFailureCode = {}));
var IpamResourceType;
(function (IpamResourceType) {
    IpamResourceType["eip"] = "eip";
    IpamResourceType["ipv6_pool"] = "ipv6-pool";
    IpamResourceType["public_ipv4_pool"] = "public-ipv4-pool";
    IpamResourceType["subnet"] = "subnet";
    IpamResourceType["vpc"] = "vpc";
})(IpamResourceType = exports.IpamResourceType || (exports.IpamResourceType = {}));
var IpamManagementState;
(function (IpamManagementState) {
    IpamManagementState["ignored"] = "ignored";
    IpamManagementState["managed"] = "managed";
    IpamManagementState["unmanaged"] = "unmanaged";
})(IpamManagementState = exports.IpamManagementState || (exports.IpamManagementState = {}));
const VolumeStatusDetailsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VolumeStatusDetailsFilterSensitiveLog = VolumeStatusDetailsFilterSensitiveLog;
const VolumeStatusInfoFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VolumeStatusInfoFilterSensitiveLog = VolumeStatusInfoFilterSensitiveLog;
const VolumeStatusItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VolumeStatusItemFilterSensitiveLog = VolumeStatusItemFilterSensitiveLog;
const DescribeVolumeStatusResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVolumeStatusResultFilterSensitiveLog = DescribeVolumeStatusResultFilterSensitiveLog;
const DescribeVpcAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcAttributeRequestFilterSensitiveLog = DescribeVpcAttributeRequestFilterSensitiveLog;
const DescribeVpcAttributeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcAttributeResultFilterSensitiveLog = DescribeVpcAttributeResultFilterSensitiveLog;
const DescribeVpcClassicLinkRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcClassicLinkRequestFilterSensitiveLog = DescribeVpcClassicLinkRequestFilterSensitiveLog;
const VpcClassicLinkFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VpcClassicLinkFilterSensitiveLog = VpcClassicLinkFilterSensitiveLog;
const DescribeVpcClassicLinkResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcClassicLinkResultFilterSensitiveLog = DescribeVpcClassicLinkResultFilterSensitiveLog;
const DescribeVpcClassicLinkDnsSupportRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcClassicLinkDnsSupportRequestFilterSensitiveLog = DescribeVpcClassicLinkDnsSupportRequestFilterSensitiveLog;
const ClassicLinkDnsSupportFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ClassicLinkDnsSupportFilterSensitiveLog = ClassicLinkDnsSupportFilterSensitiveLog;
const DescribeVpcClassicLinkDnsSupportResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcClassicLinkDnsSupportResultFilterSensitiveLog = DescribeVpcClassicLinkDnsSupportResultFilterSensitiveLog;
const DescribeVpcEndpointConnectionNotificationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointConnectionNotificationsRequestFilterSensitiveLog = DescribeVpcEndpointConnectionNotificationsRequestFilterSensitiveLog;
const DescribeVpcEndpointConnectionNotificationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointConnectionNotificationsResultFilterSensitiveLog = DescribeVpcEndpointConnectionNotificationsResultFilterSensitiveLog;
const DescribeVpcEndpointConnectionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointConnectionsRequestFilterSensitiveLog = DescribeVpcEndpointConnectionsRequestFilterSensitiveLog;
const VpcEndpointConnectionFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VpcEndpointConnectionFilterSensitiveLog = VpcEndpointConnectionFilterSensitiveLog;
const DescribeVpcEndpointConnectionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointConnectionsResultFilterSensitiveLog = DescribeVpcEndpointConnectionsResultFilterSensitiveLog;
const DescribeVpcEndpointsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointsRequestFilterSensitiveLog = DescribeVpcEndpointsRequestFilterSensitiveLog;
const DescribeVpcEndpointsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointsResultFilterSensitiveLog = DescribeVpcEndpointsResultFilterSensitiveLog;
const DescribeVpcEndpointServiceConfigurationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointServiceConfigurationsRequestFilterSensitiveLog = DescribeVpcEndpointServiceConfigurationsRequestFilterSensitiveLog;
const DescribeVpcEndpointServiceConfigurationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointServiceConfigurationsResultFilterSensitiveLog = DescribeVpcEndpointServiceConfigurationsResultFilterSensitiveLog;
const DescribeVpcEndpointServicePermissionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointServicePermissionsRequestFilterSensitiveLog = DescribeVpcEndpointServicePermissionsRequestFilterSensitiveLog;
const DescribeVpcEndpointServicePermissionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointServicePermissionsResultFilterSensitiveLog = DescribeVpcEndpointServicePermissionsResultFilterSensitiveLog;
const DescribeVpcEndpointServicesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointServicesRequestFilterSensitiveLog = DescribeVpcEndpointServicesRequestFilterSensitiveLog;
const PrivateDnsDetailsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PrivateDnsDetailsFilterSensitiveLog = PrivateDnsDetailsFilterSensitiveLog;
const ServiceDetailFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ServiceDetailFilterSensitiveLog = ServiceDetailFilterSensitiveLog;
const DescribeVpcEndpointServicesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcEndpointServicesResultFilterSensitiveLog = DescribeVpcEndpointServicesResultFilterSensitiveLog;
const DescribeVpcPeeringConnectionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcPeeringConnectionsRequestFilterSensitiveLog = DescribeVpcPeeringConnectionsRequestFilterSensitiveLog;
const DescribeVpcPeeringConnectionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcPeeringConnectionsResultFilterSensitiveLog = DescribeVpcPeeringConnectionsResultFilterSensitiveLog;
const DescribeVpcsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcsRequestFilterSensitiveLog = DescribeVpcsRequestFilterSensitiveLog;
const DescribeVpcsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpcsResultFilterSensitiveLog = DescribeVpcsResultFilterSensitiveLog;
const DescribeVpnConnectionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpnConnectionsRequestFilterSensitiveLog = DescribeVpnConnectionsRequestFilterSensitiveLog;
const DescribeVpnConnectionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpnConnectionsResultFilterSensitiveLog = DescribeVpnConnectionsResultFilterSensitiveLog;
const DescribeVpnGatewaysRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpnGatewaysRequestFilterSensitiveLog = DescribeVpnGatewaysRequestFilterSensitiveLog;
const DescribeVpnGatewaysResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DescribeVpnGatewaysResultFilterSensitiveLog = DescribeVpnGatewaysResultFilterSensitiveLog;
const DetachClassicLinkVpcRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DetachClassicLinkVpcRequestFilterSensitiveLog = DetachClassicLinkVpcRequestFilterSensitiveLog;
const DetachClassicLinkVpcResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DetachClassicLinkVpcResultFilterSensitiveLog = DetachClassicLinkVpcResultFilterSensitiveLog;
const DetachInternetGatewayRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DetachInternetGatewayRequestFilterSensitiveLog = DetachInternetGatewayRequestFilterSensitiveLog;
const DetachNetworkInterfaceRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DetachNetworkInterfaceRequestFilterSensitiveLog = DetachNetworkInterfaceRequestFilterSensitiveLog;
const DetachVerifiedAccessTrustProviderRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DetachVerifiedAccessTrustProviderRequestFilterSensitiveLog = DetachVerifiedAccessTrustProviderRequestFilterSensitiveLog;
const DetachVerifiedAccessTrustProviderResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DetachVerifiedAccessTrustProviderResultFilterSensitiveLog = DetachVerifiedAccessTrustProviderResultFilterSensitiveLog;
const DetachVolumeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DetachVolumeRequestFilterSensitiveLog = DetachVolumeRequestFilterSensitiveLog;
const DetachVpnGatewayRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DetachVpnGatewayRequestFilterSensitiveLog = DetachVpnGatewayRequestFilterSensitiveLog;
const DisableAddressTransferRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableAddressTransferRequestFilterSensitiveLog = DisableAddressTransferRequestFilterSensitiveLog;
const DisableAddressTransferResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableAddressTransferResultFilterSensitiveLog = DisableAddressTransferResultFilterSensitiveLog;
const DisableAwsNetworkPerformanceMetricSubscriptionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableAwsNetworkPerformanceMetricSubscriptionRequestFilterSensitiveLog = DisableAwsNetworkPerformanceMetricSubscriptionRequestFilterSensitiveLog;
const DisableAwsNetworkPerformanceMetricSubscriptionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableAwsNetworkPerformanceMetricSubscriptionResultFilterSensitiveLog = DisableAwsNetworkPerformanceMetricSubscriptionResultFilterSensitiveLog;
const DisableEbsEncryptionByDefaultRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableEbsEncryptionByDefaultRequestFilterSensitiveLog = DisableEbsEncryptionByDefaultRequestFilterSensitiveLog;
const DisableEbsEncryptionByDefaultResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableEbsEncryptionByDefaultResultFilterSensitiveLog = DisableEbsEncryptionByDefaultResultFilterSensitiveLog;
const DisableFastLaunchRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableFastLaunchRequestFilterSensitiveLog = DisableFastLaunchRequestFilterSensitiveLog;
const DisableFastLaunchResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableFastLaunchResultFilterSensitiveLog = DisableFastLaunchResultFilterSensitiveLog;
const DisableFastSnapshotRestoresRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableFastSnapshotRestoresRequestFilterSensitiveLog = DisableFastSnapshotRestoresRequestFilterSensitiveLog;
const DisableFastSnapshotRestoreSuccessItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableFastSnapshotRestoreSuccessItemFilterSensitiveLog = DisableFastSnapshotRestoreSuccessItemFilterSensitiveLog;
const DisableFastSnapshotRestoreStateErrorFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableFastSnapshotRestoreStateErrorFilterSensitiveLog = DisableFastSnapshotRestoreStateErrorFilterSensitiveLog;
const DisableFastSnapshotRestoreStateErrorItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableFastSnapshotRestoreStateErrorItemFilterSensitiveLog = DisableFastSnapshotRestoreStateErrorItemFilterSensitiveLog;
const DisableFastSnapshotRestoreErrorItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableFastSnapshotRestoreErrorItemFilterSensitiveLog = DisableFastSnapshotRestoreErrorItemFilterSensitiveLog;
const DisableFastSnapshotRestoresResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableFastSnapshotRestoresResultFilterSensitiveLog = DisableFastSnapshotRestoresResultFilterSensitiveLog;
const DisableImageDeprecationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableImageDeprecationRequestFilterSensitiveLog = DisableImageDeprecationRequestFilterSensitiveLog;
const DisableImageDeprecationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableImageDeprecationResultFilterSensitiveLog = DisableImageDeprecationResultFilterSensitiveLog;
const DisableIpamOrganizationAdminAccountRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableIpamOrganizationAdminAccountRequestFilterSensitiveLog = DisableIpamOrganizationAdminAccountRequestFilterSensitiveLog;
const DisableIpamOrganizationAdminAccountResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableIpamOrganizationAdminAccountResultFilterSensitiveLog = DisableIpamOrganizationAdminAccountResultFilterSensitiveLog;
const DisableSerialConsoleAccessRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableSerialConsoleAccessRequestFilterSensitiveLog = DisableSerialConsoleAccessRequestFilterSensitiveLog;
const DisableSerialConsoleAccessResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableSerialConsoleAccessResultFilterSensitiveLog = DisableSerialConsoleAccessResultFilterSensitiveLog;
const DisableTransitGatewayRouteTablePropagationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableTransitGatewayRouteTablePropagationRequestFilterSensitiveLog = DisableTransitGatewayRouteTablePropagationRequestFilterSensitiveLog;
const TransitGatewayPropagationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayPropagationFilterSensitiveLog = TransitGatewayPropagationFilterSensitiveLog;
const DisableTransitGatewayRouteTablePropagationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableTransitGatewayRouteTablePropagationResultFilterSensitiveLog = DisableTransitGatewayRouteTablePropagationResultFilterSensitiveLog;
const DisableVgwRoutePropagationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableVgwRoutePropagationRequestFilterSensitiveLog = DisableVgwRoutePropagationRequestFilterSensitiveLog;
const DisableVpcClassicLinkRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableVpcClassicLinkRequestFilterSensitiveLog = DisableVpcClassicLinkRequestFilterSensitiveLog;
const DisableVpcClassicLinkResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableVpcClassicLinkResultFilterSensitiveLog = DisableVpcClassicLinkResultFilterSensitiveLog;
const DisableVpcClassicLinkDnsSupportRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableVpcClassicLinkDnsSupportRequestFilterSensitiveLog = DisableVpcClassicLinkDnsSupportRequestFilterSensitiveLog;
const DisableVpcClassicLinkDnsSupportResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisableVpcClassicLinkDnsSupportResultFilterSensitiveLog = DisableVpcClassicLinkDnsSupportResultFilterSensitiveLog;
const DisassociateAddressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateAddressRequestFilterSensitiveLog = DisassociateAddressRequestFilterSensitiveLog;
const DisassociateClientVpnTargetNetworkRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateClientVpnTargetNetworkRequestFilterSensitiveLog = DisassociateClientVpnTargetNetworkRequestFilterSensitiveLog;
const DisassociateClientVpnTargetNetworkResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateClientVpnTargetNetworkResultFilterSensitiveLog = DisassociateClientVpnTargetNetworkResultFilterSensitiveLog;
const DisassociateEnclaveCertificateIamRoleRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateEnclaveCertificateIamRoleRequestFilterSensitiveLog = DisassociateEnclaveCertificateIamRoleRequestFilterSensitiveLog;
const DisassociateEnclaveCertificateIamRoleResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateEnclaveCertificateIamRoleResultFilterSensitiveLog = DisassociateEnclaveCertificateIamRoleResultFilterSensitiveLog;
const DisassociateIamInstanceProfileRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateIamInstanceProfileRequestFilterSensitiveLog = DisassociateIamInstanceProfileRequestFilterSensitiveLog;
const DisassociateIamInstanceProfileResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateIamInstanceProfileResultFilterSensitiveLog = DisassociateIamInstanceProfileResultFilterSensitiveLog;
const InstanceEventWindowDisassociationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceEventWindowDisassociationRequestFilterSensitiveLog = InstanceEventWindowDisassociationRequestFilterSensitiveLog;
const DisassociateInstanceEventWindowRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateInstanceEventWindowRequestFilterSensitiveLog = DisassociateInstanceEventWindowRequestFilterSensitiveLog;
const DisassociateInstanceEventWindowResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateInstanceEventWindowResultFilterSensitiveLog = DisassociateInstanceEventWindowResultFilterSensitiveLog;
const DisassociateIpamResourceDiscoveryRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateIpamResourceDiscoveryRequestFilterSensitiveLog = DisassociateIpamResourceDiscoveryRequestFilterSensitiveLog;
const DisassociateIpamResourceDiscoveryResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateIpamResourceDiscoveryResultFilterSensitiveLog = DisassociateIpamResourceDiscoveryResultFilterSensitiveLog;
const DisassociateNatGatewayAddressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateNatGatewayAddressRequestFilterSensitiveLog = DisassociateNatGatewayAddressRequestFilterSensitiveLog;
const DisassociateNatGatewayAddressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateNatGatewayAddressResultFilterSensitiveLog = DisassociateNatGatewayAddressResultFilterSensitiveLog;
const DisassociateRouteTableRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateRouteTableRequestFilterSensitiveLog = DisassociateRouteTableRequestFilterSensitiveLog;
const DisassociateSubnetCidrBlockRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateSubnetCidrBlockRequestFilterSensitiveLog = DisassociateSubnetCidrBlockRequestFilterSensitiveLog;
const DisassociateSubnetCidrBlockResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateSubnetCidrBlockResultFilterSensitiveLog = DisassociateSubnetCidrBlockResultFilterSensitiveLog;
const DisassociateTransitGatewayMulticastDomainRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateTransitGatewayMulticastDomainRequestFilterSensitiveLog = DisassociateTransitGatewayMulticastDomainRequestFilterSensitiveLog;
const DisassociateTransitGatewayMulticastDomainResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateTransitGatewayMulticastDomainResultFilterSensitiveLog = DisassociateTransitGatewayMulticastDomainResultFilterSensitiveLog;
const DisassociateTransitGatewayPolicyTableRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateTransitGatewayPolicyTableRequestFilterSensitiveLog = DisassociateTransitGatewayPolicyTableRequestFilterSensitiveLog;
const DisassociateTransitGatewayPolicyTableResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateTransitGatewayPolicyTableResultFilterSensitiveLog = DisassociateTransitGatewayPolicyTableResultFilterSensitiveLog;
const DisassociateTransitGatewayRouteTableRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateTransitGatewayRouteTableRequestFilterSensitiveLog = DisassociateTransitGatewayRouteTableRequestFilterSensitiveLog;
const DisassociateTransitGatewayRouteTableResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateTransitGatewayRouteTableResultFilterSensitiveLog = DisassociateTransitGatewayRouteTableResultFilterSensitiveLog;
const DisassociateTrunkInterfaceRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateTrunkInterfaceRequestFilterSensitiveLog = DisassociateTrunkInterfaceRequestFilterSensitiveLog;
const DisassociateTrunkInterfaceResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateTrunkInterfaceResultFilterSensitiveLog = DisassociateTrunkInterfaceResultFilterSensitiveLog;
const DisassociateVpcCidrBlockRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateVpcCidrBlockRequestFilterSensitiveLog = DisassociateVpcCidrBlockRequestFilterSensitiveLog;
const DisassociateVpcCidrBlockResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DisassociateVpcCidrBlockResultFilterSensitiveLog = DisassociateVpcCidrBlockResultFilterSensitiveLog;
const EnableAddressTransferRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableAddressTransferRequestFilterSensitiveLog = EnableAddressTransferRequestFilterSensitiveLog;
const EnableAddressTransferResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableAddressTransferResultFilterSensitiveLog = EnableAddressTransferResultFilterSensitiveLog;
const EnableAwsNetworkPerformanceMetricSubscriptionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableAwsNetworkPerformanceMetricSubscriptionRequestFilterSensitiveLog = EnableAwsNetworkPerformanceMetricSubscriptionRequestFilterSensitiveLog;
const EnableAwsNetworkPerformanceMetricSubscriptionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableAwsNetworkPerformanceMetricSubscriptionResultFilterSensitiveLog = EnableAwsNetworkPerformanceMetricSubscriptionResultFilterSensitiveLog;
const EnableEbsEncryptionByDefaultRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableEbsEncryptionByDefaultRequestFilterSensitiveLog = EnableEbsEncryptionByDefaultRequestFilterSensitiveLog;
const EnableEbsEncryptionByDefaultResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableEbsEncryptionByDefaultResultFilterSensitiveLog = EnableEbsEncryptionByDefaultResultFilterSensitiveLog;
const FastLaunchLaunchTemplateSpecificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.FastLaunchLaunchTemplateSpecificationRequestFilterSensitiveLog = FastLaunchLaunchTemplateSpecificationRequestFilterSensitiveLog;
const FastLaunchSnapshotConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.FastLaunchSnapshotConfigurationRequestFilterSensitiveLog = FastLaunchSnapshotConfigurationRequestFilterSensitiveLog;
const EnableFastLaunchRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableFastLaunchRequestFilterSensitiveLog = EnableFastLaunchRequestFilterSensitiveLog;
const EnableFastLaunchResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableFastLaunchResultFilterSensitiveLog = EnableFastLaunchResultFilterSensitiveLog;
const EnableFastSnapshotRestoresRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableFastSnapshotRestoresRequestFilterSensitiveLog = EnableFastSnapshotRestoresRequestFilterSensitiveLog;
const EnableFastSnapshotRestoreSuccessItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableFastSnapshotRestoreSuccessItemFilterSensitiveLog = EnableFastSnapshotRestoreSuccessItemFilterSensitiveLog;
const EnableFastSnapshotRestoreStateErrorFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableFastSnapshotRestoreStateErrorFilterSensitiveLog = EnableFastSnapshotRestoreStateErrorFilterSensitiveLog;
const EnableFastSnapshotRestoreStateErrorItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableFastSnapshotRestoreStateErrorItemFilterSensitiveLog = EnableFastSnapshotRestoreStateErrorItemFilterSensitiveLog;
const EnableFastSnapshotRestoreErrorItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableFastSnapshotRestoreErrorItemFilterSensitiveLog = EnableFastSnapshotRestoreErrorItemFilterSensitiveLog;
const EnableFastSnapshotRestoresResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableFastSnapshotRestoresResultFilterSensitiveLog = EnableFastSnapshotRestoresResultFilterSensitiveLog;
const EnableImageDeprecationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableImageDeprecationRequestFilterSensitiveLog = EnableImageDeprecationRequestFilterSensitiveLog;
const EnableImageDeprecationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableImageDeprecationResultFilterSensitiveLog = EnableImageDeprecationResultFilterSensitiveLog;
const EnableIpamOrganizationAdminAccountRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableIpamOrganizationAdminAccountRequestFilterSensitiveLog = EnableIpamOrganizationAdminAccountRequestFilterSensitiveLog;
const EnableIpamOrganizationAdminAccountResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableIpamOrganizationAdminAccountResultFilterSensitiveLog = EnableIpamOrganizationAdminAccountResultFilterSensitiveLog;
const EnableReachabilityAnalyzerOrganizationSharingRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableReachabilityAnalyzerOrganizationSharingRequestFilterSensitiveLog = EnableReachabilityAnalyzerOrganizationSharingRequestFilterSensitiveLog;
const EnableReachabilityAnalyzerOrganizationSharingResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableReachabilityAnalyzerOrganizationSharingResultFilterSensitiveLog = EnableReachabilityAnalyzerOrganizationSharingResultFilterSensitiveLog;
const EnableSerialConsoleAccessRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableSerialConsoleAccessRequestFilterSensitiveLog = EnableSerialConsoleAccessRequestFilterSensitiveLog;
const EnableSerialConsoleAccessResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableSerialConsoleAccessResultFilterSensitiveLog = EnableSerialConsoleAccessResultFilterSensitiveLog;
const EnableTransitGatewayRouteTablePropagationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableTransitGatewayRouteTablePropagationRequestFilterSensitiveLog = EnableTransitGatewayRouteTablePropagationRequestFilterSensitiveLog;
const EnableTransitGatewayRouteTablePropagationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableTransitGatewayRouteTablePropagationResultFilterSensitiveLog = EnableTransitGatewayRouteTablePropagationResultFilterSensitiveLog;
const EnableVgwRoutePropagationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableVgwRoutePropagationRequestFilterSensitiveLog = EnableVgwRoutePropagationRequestFilterSensitiveLog;
const EnableVolumeIORequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableVolumeIORequestFilterSensitiveLog = EnableVolumeIORequestFilterSensitiveLog;
const EnableVpcClassicLinkRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableVpcClassicLinkRequestFilterSensitiveLog = EnableVpcClassicLinkRequestFilterSensitiveLog;
const EnableVpcClassicLinkResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableVpcClassicLinkResultFilterSensitiveLog = EnableVpcClassicLinkResultFilterSensitiveLog;
const EnableVpcClassicLinkDnsSupportRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableVpcClassicLinkDnsSupportRequestFilterSensitiveLog = EnableVpcClassicLinkDnsSupportRequestFilterSensitiveLog;
const EnableVpcClassicLinkDnsSupportResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnableVpcClassicLinkDnsSupportResultFilterSensitiveLog = EnableVpcClassicLinkDnsSupportResultFilterSensitiveLog;
const ExportClientVpnClientCertificateRevocationListRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ExportClientVpnClientCertificateRevocationListRequestFilterSensitiveLog = ExportClientVpnClientCertificateRevocationListRequestFilterSensitiveLog;
const ClientCertificateRevocationListStatusFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ClientCertificateRevocationListStatusFilterSensitiveLog = ClientCertificateRevocationListStatusFilterSensitiveLog;
const ExportClientVpnClientCertificateRevocationListResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ExportClientVpnClientCertificateRevocationListResultFilterSensitiveLog = ExportClientVpnClientCertificateRevocationListResultFilterSensitiveLog;
const ExportClientVpnClientConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ExportClientVpnClientConfigurationRequestFilterSensitiveLog = ExportClientVpnClientConfigurationRequestFilterSensitiveLog;
const ExportClientVpnClientConfigurationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ExportClientVpnClientConfigurationResultFilterSensitiveLog = ExportClientVpnClientConfigurationResultFilterSensitiveLog;
const ExportTaskS3LocationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ExportTaskS3LocationRequestFilterSensitiveLog = ExportTaskS3LocationRequestFilterSensitiveLog;
const ExportImageRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ExportImageRequestFilterSensitiveLog = ExportImageRequestFilterSensitiveLog;
const ExportImageResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ExportImageResultFilterSensitiveLog = ExportImageResultFilterSensitiveLog;
const ExportTransitGatewayRoutesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ExportTransitGatewayRoutesRequestFilterSensitiveLog = ExportTransitGatewayRoutesRequestFilterSensitiveLog;
const ExportTransitGatewayRoutesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ExportTransitGatewayRoutesResultFilterSensitiveLog = ExportTransitGatewayRoutesResultFilterSensitiveLog;
const GetAssociatedEnclaveCertificateIamRolesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetAssociatedEnclaveCertificateIamRolesRequestFilterSensitiveLog = GetAssociatedEnclaveCertificateIamRolesRequestFilterSensitiveLog;
const AssociatedRoleFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.AssociatedRoleFilterSensitiveLog = AssociatedRoleFilterSensitiveLog;
const GetAssociatedEnclaveCertificateIamRolesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetAssociatedEnclaveCertificateIamRolesResultFilterSensitiveLog = GetAssociatedEnclaveCertificateIamRolesResultFilterSensitiveLog;
const GetAssociatedIpv6PoolCidrsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetAssociatedIpv6PoolCidrsRequestFilterSensitiveLog = GetAssociatedIpv6PoolCidrsRequestFilterSensitiveLog;
const Ipv6CidrAssociationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.Ipv6CidrAssociationFilterSensitiveLog = Ipv6CidrAssociationFilterSensitiveLog;
const GetAssociatedIpv6PoolCidrsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetAssociatedIpv6PoolCidrsResultFilterSensitiveLog = GetAssociatedIpv6PoolCidrsResultFilterSensitiveLog;
const DataQueryFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DataQueryFilterSensitiveLog = DataQueryFilterSensitiveLog;
const GetAwsNetworkPerformanceDataRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetAwsNetworkPerformanceDataRequestFilterSensitiveLog = GetAwsNetworkPerformanceDataRequestFilterSensitiveLog;
const MetricPointFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.MetricPointFilterSensitiveLog = MetricPointFilterSensitiveLog;
const DataResponseFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DataResponseFilterSensitiveLog = DataResponseFilterSensitiveLog;
const GetAwsNetworkPerformanceDataResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetAwsNetworkPerformanceDataResultFilterSensitiveLog = GetAwsNetworkPerformanceDataResultFilterSensitiveLog;
const GetCapacityReservationUsageRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetCapacityReservationUsageRequestFilterSensitiveLog = GetCapacityReservationUsageRequestFilterSensitiveLog;
const InstanceUsageFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceUsageFilterSensitiveLog = InstanceUsageFilterSensitiveLog;
const GetCapacityReservationUsageResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetCapacityReservationUsageResultFilterSensitiveLog = GetCapacityReservationUsageResultFilterSensitiveLog;
const GetCoipPoolUsageRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetCoipPoolUsageRequestFilterSensitiveLog = GetCoipPoolUsageRequestFilterSensitiveLog;
const CoipAddressUsageFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.CoipAddressUsageFilterSensitiveLog = CoipAddressUsageFilterSensitiveLog;
const GetCoipPoolUsageResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetCoipPoolUsageResultFilterSensitiveLog = GetCoipPoolUsageResultFilterSensitiveLog;
const GetConsoleOutputRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetConsoleOutputRequestFilterSensitiveLog = GetConsoleOutputRequestFilterSensitiveLog;
const GetConsoleOutputResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetConsoleOutputResultFilterSensitiveLog = GetConsoleOutputResultFilterSensitiveLog;
const GetConsoleScreenshotRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetConsoleScreenshotRequestFilterSensitiveLog = GetConsoleScreenshotRequestFilterSensitiveLog;
const GetConsoleScreenshotResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetConsoleScreenshotResultFilterSensitiveLog = GetConsoleScreenshotResultFilterSensitiveLog;
const GetDefaultCreditSpecificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetDefaultCreditSpecificationRequestFilterSensitiveLog = GetDefaultCreditSpecificationRequestFilterSensitiveLog;
const InstanceFamilyCreditSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceFamilyCreditSpecificationFilterSensitiveLog = InstanceFamilyCreditSpecificationFilterSensitiveLog;
const GetDefaultCreditSpecificationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetDefaultCreditSpecificationResultFilterSensitiveLog = GetDefaultCreditSpecificationResultFilterSensitiveLog;
const GetEbsDefaultKmsKeyIdRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetEbsDefaultKmsKeyIdRequestFilterSensitiveLog = GetEbsDefaultKmsKeyIdRequestFilterSensitiveLog;
const GetEbsDefaultKmsKeyIdResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetEbsDefaultKmsKeyIdResultFilterSensitiveLog = GetEbsDefaultKmsKeyIdResultFilterSensitiveLog;
const GetEbsEncryptionByDefaultRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetEbsEncryptionByDefaultRequestFilterSensitiveLog = GetEbsEncryptionByDefaultRequestFilterSensitiveLog;
const GetEbsEncryptionByDefaultResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetEbsEncryptionByDefaultResultFilterSensitiveLog = GetEbsEncryptionByDefaultResultFilterSensitiveLog;
const AthenaIntegrationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.AthenaIntegrationFilterSensitiveLog = AthenaIntegrationFilterSensitiveLog;
const IntegrateServicesFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.IntegrateServicesFilterSensitiveLog = IntegrateServicesFilterSensitiveLog;
const GetFlowLogsIntegrationTemplateRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetFlowLogsIntegrationTemplateRequestFilterSensitiveLog = GetFlowLogsIntegrationTemplateRequestFilterSensitiveLog;
const GetFlowLogsIntegrationTemplateResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetFlowLogsIntegrationTemplateResultFilterSensitiveLog = GetFlowLogsIntegrationTemplateResultFilterSensitiveLog;
const GetGroupsForCapacityReservationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetGroupsForCapacityReservationRequestFilterSensitiveLog = GetGroupsForCapacityReservationRequestFilterSensitiveLog;
const CapacityReservationGroupFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.CapacityReservationGroupFilterSensitiveLog = CapacityReservationGroupFilterSensitiveLog;
const GetGroupsForCapacityReservationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetGroupsForCapacityReservationResultFilterSensitiveLog = GetGroupsForCapacityReservationResultFilterSensitiveLog;
const GetHostReservationPurchasePreviewRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetHostReservationPurchasePreviewRequestFilterSensitiveLog = GetHostReservationPurchasePreviewRequestFilterSensitiveLog;
const PurchaseFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PurchaseFilterSensitiveLog = PurchaseFilterSensitiveLog;
const GetHostReservationPurchasePreviewResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetHostReservationPurchasePreviewResultFilterSensitiveLog = GetHostReservationPurchasePreviewResultFilterSensitiveLog;
const GetInstanceTypesFromInstanceRequirementsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetInstanceTypesFromInstanceRequirementsRequestFilterSensitiveLog = GetInstanceTypesFromInstanceRequirementsRequestFilterSensitiveLog;
const InstanceTypeInfoFromInstanceRequirementsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceTypeInfoFromInstanceRequirementsFilterSensitiveLog = InstanceTypeInfoFromInstanceRequirementsFilterSensitiveLog;
const GetInstanceTypesFromInstanceRequirementsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetInstanceTypesFromInstanceRequirementsResultFilterSensitiveLog = GetInstanceTypesFromInstanceRequirementsResultFilterSensitiveLog;
const GetInstanceUefiDataRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetInstanceUefiDataRequestFilterSensitiveLog = GetInstanceUefiDataRequestFilterSensitiveLog;
const GetInstanceUefiDataResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetInstanceUefiDataResultFilterSensitiveLog = GetInstanceUefiDataResultFilterSensitiveLog;
const GetIpamAddressHistoryRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamAddressHistoryRequestFilterSensitiveLog = GetIpamAddressHistoryRequestFilterSensitiveLog;
const IpamAddressHistoryRecordFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.IpamAddressHistoryRecordFilterSensitiveLog = IpamAddressHistoryRecordFilterSensitiveLog;
const GetIpamAddressHistoryResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamAddressHistoryResultFilterSensitiveLog = GetIpamAddressHistoryResultFilterSensitiveLog;
const GetIpamDiscoveredAccountsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamDiscoveredAccountsRequestFilterSensitiveLog = GetIpamDiscoveredAccountsRequestFilterSensitiveLog;
const IpamDiscoveryFailureReasonFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.IpamDiscoveryFailureReasonFilterSensitiveLog = IpamDiscoveryFailureReasonFilterSensitiveLog;
const IpamDiscoveredAccountFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.IpamDiscoveredAccountFilterSensitiveLog = IpamDiscoveredAccountFilterSensitiveLog;
const GetIpamDiscoveredAccountsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamDiscoveredAccountsResultFilterSensitiveLog = GetIpamDiscoveredAccountsResultFilterSensitiveLog;
const GetIpamDiscoveredResourceCidrsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamDiscoveredResourceCidrsRequestFilterSensitiveLog = GetIpamDiscoveredResourceCidrsRequestFilterSensitiveLog;
const IpamDiscoveredResourceCidrFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.IpamDiscoveredResourceCidrFilterSensitiveLog = IpamDiscoveredResourceCidrFilterSensitiveLog;
const GetIpamDiscoveredResourceCidrsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamDiscoveredResourceCidrsResultFilterSensitiveLog = GetIpamDiscoveredResourceCidrsResultFilterSensitiveLog;
const GetIpamPoolAllocationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamPoolAllocationsRequestFilterSensitiveLog = GetIpamPoolAllocationsRequestFilterSensitiveLog;
const GetIpamPoolAllocationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamPoolAllocationsResultFilterSensitiveLog = GetIpamPoolAllocationsResultFilterSensitiveLog;
const GetIpamPoolCidrsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamPoolCidrsRequestFilterSensitiveLog = GetIpamPoolCidrsRequestFilterSensitiveLog;
const GetIpamPoolCidrsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamPoolCidrsResultFilterSensitiveLog = GetIpamPoolCidrsResultFilterSensitiveLog;
const GetIpamResourceCidrsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamResourceCidrsRequestFilterSensitiveLog = GetIpamResourceCidrsRequestFilterSensitiveLog;
const IpamResourceCidrFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.IpamResourceCidrFilterSensitiveLog = IpamResourceCidrFilterSensitiveLog;
const GetIpamResourceCidrsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetIpamResourceCidrsResultFilterSensitiveLog = GetIpamResourceCidrsResultFilterSensitiveLog;
const GetLaunchTemplateDataRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetLaunchTemplateDataRequestFilterSensitiveLog = GetLaunchTemplateDataRequestFilterSensitiveLog;
const GetLaunchTemplateDataResultFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.LaunchTemplateData && {
        LaunchTemplateData: (0, models_1_1.ResponseLaunchTemplateDataFilterSensitiveLog)(obj.LaunchTemplateData),
    }),
});
exports.GetLaunchTemplateDataResultFilterSensitiveLog = GetLaunchTemplateDataResultFilterSensitiveLog;
const GetManagedPrefixListAssociationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetManagedPrefixListAssociationsRequestFilterSensitiveLog = GetManagedPrefixListAssociationsRequestFilterSensitiveLog;
const PrefixListAssociationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PrefixListAssociationFilterSensitiveLog = PrefixListAssociationFilterSensitiveLog;
const GetManagedPrefixListAssociationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetManagedPrefixListAssociationsResultFilterSensitiveLog = GetManagedPrefixListAssociationsResultFilterSensitiveLog;
const GetManagedPrefixListEntriesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetManagedPrefixListEntriesRequestFilterSensitiveLog = GetManagedPrefixListEntriesRequestFilterSensitiveLog;
const PrefixListEntryFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PrefixListEntryFilterSensitiveLog = PrefixListEntryFilterSensitiveLog;
const GetManagedPrefixListEntriesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetManagedPrefixListEntriesResultFilterSensitiveLog = GetManagedPrefixListEntriesResultFilterSensitiveLog;
const GetNetworkInsightsAccessScopeAnalysisFindingsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetNetworkInsightsAccessScopeAnalysisFindingsRequestFilterSensitiveLog = GetNetworkInsightsAccessScopeAnalysisFindingsRequestFilterSensitiveLog;
const GetNetworkInsightsAccessScopeAnalysisFindingsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetNetworkInsightsAccessScopeAnalysisFindingsResultFilterSensitiveLog = GetNetworkInsightsAccessScopeAnalysisFindingsResultFilterSensitiveLog;
const GetNetworkInsightsAccessScopeContentRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetNetworkInsightsAccessScopeContentRequestFilterSensitiveLog = GetNetworkInsightsAccessScopeContentRequestFilterSensitiveLog;
const GetNetworkInsightsAccessScopeContentResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetNetworkInsightsAccessScopeContentResultFilterSensitiveLog = GetNetworkInsightsAccessScopeContentResultFilterSensitiveLog;
const GetPasswordDataRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetPasswordDataRequestFilterSensitiveLog = GetPasswordDataRequestFilterSensitiveLog;
const GetPasswordDataResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetPasswordDataResultFilterSensitiveLog = GetPasswordDataResultFilterSensitiveLog;
const GetReservedInstancesExchangeQuoteRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetReservedInstancesExchangeQuoteRequestFilterSensitiveLog = GetReservedInstancesExchangeQuoteRequestFilterSensitiveLog;
const ReservationValueFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReservationValueFilterSensitiveLog = ReservationValueFilterSensitiveLog;
const ReservedInstanceReservationValueFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReservedInstanceReservationValueFilterSensitiveLog = ReservedInstanceReservationValueFilterSensitiveLog;
const TargetConfigurationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TargetConfigurationFilterSensitiveLog = TargetConfigurationFilterSensitiveLog;
const TargetReservationValueFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TargetReservationValueFilterSensitiveLog = TargetReservationValueFilterSensitiveLog;
const GetReservedInstancesExchangeQuoteResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetReservedInstancesExchangeQuoteResultFilterSensitiveLog = GetReservedInstancesExchangeQuoteResultFilterSensitiveLog;
const GetSerialConsoleAccessStatusRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetSerialConsoleAccessStatusRequestFilterSensitiveLog = GetSerialConsoleAccessStatusRequestFilterSensitiveLog;
const GetSerialConsoleAccessStatusResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetSerialConsoleAccessStatusResultFilterSensitiveLog = GetSerialConsoleAccessStatusResultFilterSensitiveLog;
const InstanceRequirementsWithMetadataRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceRequirementsWithMetadataRequestFilterSensitiveLog = InstanceRequirementsWithMetadataRequestFilterSensitiveLog;
const GetSpotPlacementScoresRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetSpotPlacementScoresRequestFilterSensitiveLog = GetSpotPlacementScoresRequestFilterSensitiveLog;
const SpotPlacementScoreFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SpotPlacementScoreFilterSensitiveLog = SpotPlacementScoreFilterSensitiveLog;
const GetSpotPlacementScoresResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetSpotPlacementScoresResultFilterSensitiveLog = GetSpotPlacementScoresResultFilterSensitiveLog;
const GetSubnetCidrReservationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetSubnetCidrReservationsRequestFilterSensitiveLog = GetSubnetCidrReservationsRequestFilterSensitiveLog;
const GetSubnetCidrReservationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetSubnetCidrReservationsResultFilterSensitiveLog = GetSubnetCidrReservationsResultFilterSensitiveLog;
const GetTransitGatewayAttachmentPropagationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayAttachmentPropagationsRequestFilterSensitiveLog = GetTransitGatewayAttachmentPropagationsRequestFilterSensitiveLog;
const TransitGatewayAttachmentPropagationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayAttachmentPropagationFilterSensitiveLog = TransitGatewayAttachmentPropagationFilterSensitiveLog;
const GetTransitGatewayAttachmentPropagationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayAttachmentPropagationsResultFilterSensitiveLog = GetTransitGatewayAttachmentPropagationsResultFilterSensitiveLog;
const GetTransitGatewayMulticastDomainAssociationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayMulticastDomainAssociationsRequestFilterSensitiveLog = GetTransitGatewayMulticastDomainAssociationsRequestFilterSensitiveLog;
const TransitGatewayMulticastDomainAssociationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayMulticastDomainAssociationFilterSensitiveLog = TransitGatewayMulticastDomainAssociationFilterSensitiveLog;
const GetTransitGatewayMulticastDomainAssociationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayMulticastDomainAssociationsResultFilterSensitiveLog = GetTransitGatewayMulticastDomainAssociationsResultFilterSensitiveLog;
const GetTransitGatewayPolicyTableAssociationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayPolicyTableAssociationsRequestFilterSensitiveLog = GetTransitGatewayPolicyTableAssociationsRequestFilterSensitiveLog;
const GetTransitGatewayPolicyTableAssociationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayPolicyTableAssociationsResultFilterSensitiveLog = GetTransitGatewayPolicyTableAssociationsResultFilterSensitiveLog;
const GetTransitGatewayPolicyTableEntriesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayPolicyTableEntriesRequestFilterSensitiveLog = GetTransitGatewayPolicyTableEntriesRequestFilterSensitiveLog;
const TransitGatewayPolicyRuleMetaDataFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayPolicyRuleMetaDataFilterSensitiveLog = TransitGatewayPolicyRuleMetaDataFilterSensitiveLog;
const TransitGatewayPolicyRuleFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayPolicyRuleFilterSensitiveLog = TransitGatewayPolicyRuleFilterSensitiveLog;
const TransitGatewayPolicyTableEntryFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayPolicyTableEntryFilterSensitiveLog = TransitGatewayPolicyTableEntryFilterSensitiveLog;
const GetTransitGatewayPolicyTableEntriesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayPolicyTableEntriesResultFilterSensitiveLog = GetTransitGatewayPolicyTableEntriesResultFilterSensitiveLog;
const GetTransitGatewayPrefixListReferencesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayPrefixListReferencesRequestFilterSensitiveLog = GetTransitGatewayPrefixListReferencesRequestFilterSensitiveLog;
const GetTransitGatewayPrefixListReferencesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayPrefixListReferencesResultFilterSensitiveLog = GetTransitGatewayPrefixListReferencesResultFilterSensitiveLog;
const GetTransitGatewayRouteTableAssociationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayRouteTableAssociationsRequestFilterSensitiveLog = GetTransitGatewayRouteTableAssociationsRequestFilterSensitiveLog;
const TransitGatewayRouteTableAssociationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayRouteTableAssociationFilterSensitiveLog = TransitGatewayRouteTableAssociationFilterSensitiveLog;
const GetTransitGatewayRouteTableAssociationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayRouteTableAssociationsResultFilterSensitiveLog = GetTransitGatewayRouteTableAssociationsResultFilterSensitiveLog;
const GetTransitGatewayRouteTablePropagationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayRouteTablePropagationsRequestFilterSensitiveLog = GetTransitGatewayRouteTablePropagationsRequestFilterSensitiveLog;
const TransitGatewayRouteTablePropagationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayRouteTablePropagationFilterSensitiveLog = TransitGatewayRouteTablePropagationFilterSensitiveLog;
const GetTransitGatewayRouteTablePropagationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetTransitGatewayRouteTablePropagationsResultFilterSensitiveLog = GetTransitGatewayRouteTablePropagationsResultFilterSensitiveLog;
const GetVerifiedAccessEndpointPolicyRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetVerifiedAccessEndpointPolicyRequestFilterSensitiveLog = GetVerifiedAccessEndpointPolicyRequestFilterSensitiveLog;
const GetVerifiedAccessEndpointPolicyResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetVerifiedAccessEndpointPolicyResultFilterSensitiveLog = GetVerifiedAccessEndpointPolicyResultFilterSensitiveLog;
const GetVerifiedAccessGroupPolicyRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetVerifiedAccessGroupPolicyRequestFilterSensitiveLog = GetVerifiedAccessGroupPolicyRequestFilterSensitiveLog;
const GetVerifiedAccessGroupPolicyResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetVerifiedAccessGroupPolicyResultFilterSensitiveLog = GetVerifiedAccessGroupPolicyResultFilterSensitiveLog;
const GetVpnConnectionDeviceSampleConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetVpnConnectionDeviceSampleConfigurationRequestFilterSensitiveLog = GetVpnConnectionDeviceSampleConfigurationRequestFilterSensitiveLog;
const GetVpnConnectionDeviceSampleConfigurationResultFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.VpnConnectionDeviceSampleConfiguration && { VpnConnectionDeviceSampleConfiguration: smithy_client_1.SENSITIVE_STRING }),
});
exports.GetVpnConnectionDeviceSampleConfigurationResultFilterSensitiveLog = GetVpnConnectionDeviceSampleConfigurationResultFilterSensitiveLog;
const GetVpnConnectionDeviceTypesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetVpnConnectionDeviceTypesRequestFilterSensitiveLog = GetVpnConnectionDeviceTypesRequestFilterSensitiveLog;
const VpnConnectionDeviceTypeFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VpnConnectionDeviceTypeFilterSensitiveLog = VpnConnectionDeviceTypeFilterSensitiveLog;
const GetVpnConnectionDeviceTypesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.GetVpnConnectionDeviceTypesResultFilterSensitiveLog = GetVpnConnectionDeviceTypesResultFilterSensitiveLog;
const ImportClientVpnClientCertificateRevocationListRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportClientVpnClientCertificateRevocationListRequestFilterSensitiveLog = ImportClientVpnClientCertificateRevocationListRequestFilterSensitiveLog;
const ImportClientVpnClientCertificateRevocationListResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportClientVpnClientCertificateRevocationListResultFilterSensitiveLog = ImportClientVpnClientCertificateRevocationListResultFilterSensitiveLog;
const ClientDataFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ClientDataFilterSensitiveLog = ClientDataFilterSensitiveLog;
const UserBucketFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.UserBucketFilterSensitiveLog = UserBucketFilterSensitiveLog;
const ImageDiskContainerFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImageDiskContainerFilterSensitiveLog = ImageDiskContainerFilterSensitiveLog;
const ImportImageLicenseConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportImageLicenseConfigurationRequestFilterSensitiveLog = ImportImageLicenseConfigurationRequestFilterSensitiveLog;
const ImportImageRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportImageRequestFilterSensitiveLog = ImportImageRequestFilterSensitiveLog;
const ImportImageResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportImageResultFilterSensitiveLog = ImportImageResultFilterSensitiveLog;
const DiskImageDetailFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DiskImageDetailFilterSensitiveLog = DiskImageDetailFilterSensitiveLog;
const VolumeDetailFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VolumeDetailFilterSensitiveLog = VolumeDetailFilterSensitiveLog;
const DiskImageFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DiskImageFilterSensitiveLog = DiskImageFilterSensitiveLog;
const UserDataFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.UserDataFilterSensitiveLog = UserDataFilterSensitiveLog;
const ImportInstanceLaunchSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.UserData && { UserData: smithy_client_1.SENSITIVE_STRING }),
});
exports.ImportInstanceLaunchSpecificationFilterSensitiveLog = ImportInstanceLaunchSpecificationFilterSensitiveLog;
const ImportInstanceRequestFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.LaunchSpecification && {
        LaunchSpecification: (0, exports.ImportInstanceLaunchSpecificationFilterSensitiveLog)(obj.LaunchSpecification),
    }),
});
exports.ImportInstanceRequestFilterSensitiveLog = ImportInstanceRequestFilterSensitiveLog;
const ImportInstanceResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportInstanceResultFilterSensitiveLog = ImportInstanceResultFilterSensitiveLog;
const ImportKeyPairRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportKeyPairRequestFilterSensitiveLog = ImportKeyPairRequestFilterSensitiveLog;
const ImportKeyPairResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportKeyPairResultFilterSensitiveLog = ImportKeyPairResultFilterSensitiveLog;
const SnapshotDiskContainerFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SnapshotDiskContainerFilterSensitiveLog = SnapshotDiskContainerFilterSensitiveLog;
const ImportSnapshotRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportSnapshotRequestFilterSensitiveLog = ImportSnapshotRequestFilterSensitiveLog;
const ImportSnapshotResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportSnapshotResultFilterSensitiveLog = ImportSnapshotResultFilterSensitiveLog;
const ImportVolumeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportVolumeRequestFilterSensitiveLog = ImportVolumeRequestFilterSensitiveLog;
const ImportVolumeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImportVolumeResultFilterSensitiveLog = ImportVolumeResultFilterSensitiveLog;
const ListImagesInRecycleBinRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ListImagesInRecycleBinRequestFilterSensitiveLog = ListImagesInRecycleBinRequestFilterSensitiveLog;
const ImageRecycleBinInfoFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ImageRecycleBinInfoFilterSensitiveLog = ImageRecycleBinInfoFilterSensitiveLog;
const ListImagesInRecycleBinResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ListImagesInRecycleBinResultFilterSensitiveLog = ListImagesInRecycleBinResultFilterSensitiveLog;
const ListSnapshotsInRecycleBinRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ListSnapshotsInRecycleBinRequestFilterSensitiveLog = ListSnapshotsInRecycleBinRequestFilterSensitiveLog;
const SnapshotRecycleBinInfoFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SnapshotRecycleBinInfoFilterSensitiveLog = SnapshotRecycleBinInfoFilterSensitiveLog;
const ListSnapshotsInRecycleBinResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ListSnapshotsInRecycleBinResultFilterSensitiveLog = ListSnapshotsInRecycleBinResultFilterSensitiveLog;
const ModifyAddressAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyAddressAttributeRequestFilterSensitiveLog = ModifyAddressAttributeRequestFilterSensitiveLog;
const ModifyAddressAttributeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyAddressAttributeResultFilterSensitiveLog = ModifyAddressAttributeResultFilterSensitiveLog;
