// @aws-sdk/client-ec2/dist-cjs/models/models_6.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.UnsuccessfulInstanceCreditSpecificationItemErrorFilterSensitiveLog = exports.SuccessfulInstanceCreditSpecificationItemFilterSensitiveLog = exports.ModifyInstanceCreditSpecificationRequestFilterSensitiveLog = exports.InstanceCreditSpecificationRequestFilterSensitiveLog = exports.ModifyInstanceCapacityReservationAttributesResultFilterSensitiveLog = exports.ModifyInstanceCapacityReservationAttributesRequestFilterSensitiveLog = exports.CapacityReservationSpecificationFilterSensitiveLog = exports.ModifyInstanceAttributeRequestFilterSensitiveLog = exports.BlobAttributeValueFilterSensitiveLog = exports.InstanceBlockDeviceMappingSpecificationFilterSensitiveLog = exports.EbsInstanceBlockDeviceSpecificationFilterSensitiveLog = exports.ModifyImageAttributeRequestFilterSensitiveLog = exports.LaunchPermissionModificationsFilterSensitiveLog = exports.ModifyIdFormatRequestFilterSensitiveLog = exports.ModifyIdentityIdFormatRequestFilterSensitiveLog = exports.ModifyHostsResultFilterSensitiveLog = exports.ModifyHostsRequestFilterSensitiveLog = exports.ModifyFpgaImageAttributeResultFilterSensitiveLog = exports.ModifyFpgaImageAttributeRequestFilterSensitiveLog = exports.LoadPermissionModificationsFilterSensitiveLog = exports.LoadPermissionRequestFilterSensitiveLog = exports.ModifyFleetResultFilterSensitiveLog = exports.ModifyFleetRequestFilterSensitiveLog = exports.ModifyEbsDefaultKmsKeyIdResultFilterSensitiveLog = exports.ModifyEbsDefaultKmsKeyIdRequestFilterSensitiveLog = exports.ModifyDefaultCreditSpecificationResultFilterSensitiveLog = exports.ModifyDefaultCreditSpecificationRequestFilterSensitiveLog = exports.ModifyClientVpnEndpointResultFilterSensitiveLog = exports.ModifyClientVpnEndpointRequestFilterSensitiveLog = exports.DnsServersOptionsModifyStructureFilterSensitiveLog = exports.ModifyCapacityReservationFleetResultFilterSensitiveLog = exports.ModifyCapacityReservationFleetRequestFilterSensitiveLog = exports.ModifyCapacityReservationResultFilterSensitiveLog = exports.ModifyCapacityReservationRequestFilterSensitiveLog = exports.ModifyAvailabilityZoneGroupResultFilterSensitiveLog = exports.ModifyAvailabilityZoneGroupRequestFilterSensitiveLog = exports.MembershipType = exports.ResetImageAttributeName = exports.ResetFpgaImageAttributeName = exports.ReportStatusType = exports.ReportInstanceReasonCodes = exports.Status = exports.VpcTenancy = exports.TrafficMirrorSessionField = exports.TrafficMirrorFilterRuleField = exports.TargetStorageTier = exports.HostTenancy = exports.UnsuccessfulInstanceCreditSpecificationErrorCode = exports.OperationType = exports.ModifyAvailabilityZoneOptInStatus = void 0;
exports.ModifyTrafficMirrorFilterRuleRequestFilterSensitiveLog = exports.ModifyTrafficMirrorFilterNetworkServicesResultFilterSensitiveLog = exports.ModifyTrafficMirrorFilterNetworkServicesRequestFilterSensitiveLog = exports.ModifySubnetAttributeRequestFilterSensitiveLog = exports.ModifySpotFleetRequestResponseFilterSensitiveLog = exports.ModifySpotFleetRequestRequestFilterSensitiveLog = exports.ModifySnapshotTierResultFilterSensitiveLog = exports.ModifySnapshotTierRequestFilterSensitiveLog = exports.ModifySnapshotAttributeRequestFilterSensitiveLog = exports.CreateVolumePermissionModificationsFilterSensitiveLog = exports.ModifySecurityGroupRulesResultFilterSensitiveLog = exports.ModifySecurityGroupRulesRequestFilterSensitiveLog = exports.SecurityGroupRuleUpdateFilterSensitiveLog = exports.SecurityGroupRuleRequestFilterSensitiveLog = exports.ModifyReservedInstancesResultFilterSensitiveLog = exports.ModifyReservedInstancesRequestFilterSensitiveLog = exports.ModifyPrivateDnsNameOptionsResultFilterSensitiveLog = exports.ModifyPrivateDnsNameOptionsRequestFilterSensitiveLog = exports.ModifyNetworkInterfaceAttributeRequestFilterSensitiveLog = exports.NetworkInterfaceAttachmentChangesFilterSensitiveLog = exports.ModifyManagedPrefixListResultFilterSensitiveLog = exports.ModifyManagedPrefixListRequestFilterSensitiveLog = exports.RemovePrefixListEntryFilterSensitiveLog = exports.ModifyLocalGatewayRouteResultFilterSensitiveLog = exports.ModifyLocalGatewayRouteRequestFilterSensitiveLog = exports.ModifyLaunchTemplateResultFilterSensitiveLog = exports.ModifyLaunchTemplateRequestFilterSensitiveLog = exports.ModifyIpamScopeResultFilterSensitiveLog = exports.ModifyIpamScopeRequestFilterSensitiveLog = exports.ModifyIpamResourceDiscoveryResultFilterSensitiveLog = exports.ModifyIpamResourceDiscoveryRequestFilterSensitiveLog = exports.ModifyIpamResourceCidrResultFilterSensitiveLog = exports.ModifyIpamResourceCidrRequestFilterSensitiveLog = exports.ModifyIpamPoolResultFilterSensitiveLog = exports.ModifyIpamPoolRequestFilterSensitiveLog = exports.ModifyIpamResultFilterSensitiveLog = exports.ModifyIpamRequestFilterSensitiveLog = exports.RemoveIpamOperatingRegionFilterSensitiveLog = exports.ModifyInstancePlacementResultFilterSensitiveLog = exports.ModifyInstancePlacementRequestFilterSensitiveLog = exports.ModifyInstanceMetadataOptionsResultFilterSensitiveLog = exports.ModifyInstanceMetadataOptionsRequestFilterSensitiveLog = exports.ModifyInstanceMaintenanceOptionsResultFilterSensitiveLog = exports.ModifyInstanceMaintenanceOptionsRequestFilterSensitiveLog = exports.ModifyInstanceEventWindowResultFilterSensitiveLog = exports.ModifyInstanceEventWindowRequestFilterSensitiveLog = exports.ModifyInstanceEventStartTimeResultFilterSensitiveLog = exports.ModifyInstanceEventStartTimeRequestFilterSensitiveLog = exports.ModifyInstanceCreditSpecificationResultFilterSensitiveLog = exports.UnsuccessfulInstanceCreditSpecificationItemFilterSensitiveLog = void 0;
exports.ModifyVpcPeeringConnectionOptionsResultFilterSensitiveLog = exports.PeeringConnectionOptionsFilterSensitiveLog = exports.ModifyVpcPeeringConnectionOptionsRequestFilterSensitiveLog = exports.PeeringConnectionOptionsRequestFilterSensitiveLog = exports.ModifyVpcEndpointServicePermissionsResultFilterSensitiveLog = exports.ModifyVpcEndpointServicePermissionsRequestFilterSensitiveLog = exports.ModifyVpcEndpointServicePayerResponsibilityResultFilterSensitiveLog = exports.ModifyVpcEndpointServicePayerResponsibilityRequestFilterSensitiveLog = exports.ModifyVpcEndpointServiceConfigurationResultFilterSensitiveLog = exports.ModifyVpcEndpointServiceConfigurationRequestFilterSensitiveLog = exports.ModifyVpcEndpointConnectionNotificationResultFilterSensitiveLog = exports.ModifyVpcEndpointConnectionNotificationRequestFilterSensitiveLog = exports.ModifyVpcEndpointResultFilterSensitiveLog = exports.ModifyVpcEndpointRequestFilterSensitiveLog = exports.ModifyVpcAttributeRequestFilterSensitiveLog = exports.ModifyVolumeAttributeRequestFilterSensitiveLog = exports.ModifyVolumeResultFilterSensitiveLog = exports.ModifyVolumeRequestFilterSensitiveLog = exports.ModifyVerifiedAccessTrustProviderResultFilterSensitiveLog = exports.ModifyVerifiedAccessTrustProviderRequestFilterSensitiveLog = exports.ModifyVerifiedAccessTrustProviderOidcOptionsFilterSensitiveLog = exports.ModifyVerifiedAccessInstanceLoggingConfigurationResultFilterSensitiveLog = exports.ModifyVerifiedAccessInstanceLoggingConfigurationRequestFilterSensitiveLog = exports.VerifiedAccessLogOptionsFilterSensitiveLog = exports.VerifiedAccessLogS3DestinationOptionsFilterSensitiveLog = exports.VerifiedAccessLogKinesisDataFirehoseDestinationOptionsFilterSensitiveLog = exports.VerifiedAccessLogCloudWatchLogsDestinationOptionsFilterSensitiveLog = exports.ModifyVerifiedAccessInstanceResultFilterSensitiveLog = exports.ModifyVerifiedAccessInstanceRequestFilterSensitiveLog = exports.ModifyVerifiedAccessGroupPolicyResultFilterSensitiveLog = exports.ModifyVerifiedAccessGroupPolicyRequestFilterSensitiveLog = exports.ModifyVerifiedAccessGroupResultFilterSensitiveLog = exports.ModifyVerifiedAccessGroupRequestFilterSensitiveLog = exports.ModifyVerifiedAccessEndpointPolicyResultFilterSensitiveLog = exports.ModifyVerifiedAccessEndpointPolicyRequestFilterSensitiveLog = exports.ModifyVerifiedAccessEndpointResultFilterSensitiveLog = exports.ModifyVerifiedAccessEndpointRequestFilterSensitiveLog = exports.ModifyVerifiedAccessEndpointEniOptionsFilterSensitiveLog = exports.ModifyVerifiedAccessEndpointLoadBalancerOptionsFilterSensitiveLog = exports.ModifyTransitGatewayVpcAttachmentResultFilterSensitiveLog = exports.ModifyTransitGatewayVpcAttachmentRequestFilterSensitiveLog = exports.ModifyTransitGatewayVpcAttachmentRequestOptionsFilterSensitiveLog = exports.ModifyTransitGatewayPrefixListReferenceResultFilterSensitiveLog = exports.ModifyTransitGatewayPrefixListReferenceRequestFilterSensitiveLog = exports.ModifyTransitGatewayResultFilterSensitiveLog = exports.ModifyTransitGatewayRequestFilterSensitiveLog = exports.ModifyTransitGatewayOptionsFilterSensitiveLog = exports.ModifyTrafficMirrorSessionResultFilterSensitiveLog = exports.ModifyTrafficMirrorSessionRequestFilterSensitiveLog = exports.ModifyTrafficMirrorFilterRuleResultFilterSensitiveLog = void 0;
exports.RejectTransitGatewayPeeringAttachmentResultFilterSensitiveLog = exports.RejectTransitGatewayPeeringAttachmentRequestFilterSensitiveLog = exports.RejectTransitGatewayMulticastDomainAssociationsResultFilterSensitiveLog = exports.RejectTransitGatewayMulticastDomainAssociationsRequestFilterSensitiveLog = exports.RegisterTransitGatewayMulticastGroupSourcesResultFilterSensitiveLog = exports.TransitGatewayMulticastRegisteredGroupSourcesFilterSensitiveLog = exports.RegisterTransitGatewayMulticastGroupSourcesRequestFilterSensitiveLog = exports.RegisterTransitGatewayMulticastGroupMembersResultFilterSensitiveLog = exports.TransitGatewayMulticastRegisteredGroupMembersFilterSensitiveLog = exports.RegisterTransitGatewayMulticastGroupMembersRequestFilterSensitiveLog = exports.RegisterInstanceEventNotificationAttributesResultFilterSensitiveLog = exports.RegisterInstanceEventNotificationAttributesRequestFilterSensitiveLog = exports.RegisterInstanceTagAttributeRequestFilterSensitiveLog = exports.RegisterImageResultFilterSensitiveLog = exports.RegisterImageRequestFilterSensitiveLog = exports.RebootInstancesRequestFilterSensitiveLog = exports.PurchaseScheduledInstancesResultFilterSensitiveLog = exports.PurchaseScheduledInstancesRequestFilterSensitiveLog = exports.PurchaseRequestFilterSensitiveLog = exports.PurchaseReservedInstancesOfferingResultFilterSensitiveLog = exports.PurchaseReservedInstancesOfferingRequestFilterSensitiveLog = exports.ReservedInstanceLimitPriceFilterSensitiveLog = exports.PurchaseHostReservationResultFilterSensitiveLog = exports.PurchaseHostReservationRequestFilterSensitiveLog = exports.ProvisionPublicIpv4PoolCidrResultFilterSensitiveLog = exports.ProvisionPublicIpv4PoolCidrRequestFilterSensitiveLog = exports.ProvisionIpamPoolCidrResultFilterSensitiveLog = exports.ProvisionIpamPoolCidrRequestFilterSensitiveLog = exports.IpamCidrAuthorizationContextFilterSensitiveLog = exports.ProvisionByoipCidrResultFilterSensitiveLog = exports.ProvisionByoipCidrRequestFilterSensitiveLog = exports.CidrAuthorizationContextFilterSensitiveLog = exports.MoveByoipCidrToIpamResultFilterSensitiveLog = exports.MoveByoipCidrToIpamRequestFilterSensitiveLog = exports.MoveAddressToVpcResultFilterSensitiveLog = exports.MoveAddressToVpcRequestFilterSensitiveLog = exports.MonitorInstancesResultFilterSensitiveLog = exports.InstanceMonitoringFilterSensitiveLog = exports.MonitorInstancesRequestFilterSensitiveLog = exports.ModifyVpnTunnelOptionsResultFilterSensitiveLog = exports.ModifyVpnTunnelOptionsRequestFilterSensitiveLog = exports.ModifyVpnTunnelOptionsSpecificationFilterSensitiveLog = exports.ModifyVpnTunnelCertificateResultFilterSensitiveLog = exports.ModifyVpnTunnelCertificateRequestFilterSensitiveLog = exports.ModifyVpnConnectionOptionsResultFilterSensitiveLog = exports.ModifyVpnConnectionOptionsRequestFilterSensitiveLog = exports.ModifyVpnConnectionResultFilterSensitiveLog = exports.ModifyVpnConnectionRequestFilterSensitiveLog = exports.ModifyVpcTenancyResultFilterSensitiveLog = exports.ModifyVpcTenancyRequestFilterSensitiveLog = void 0;
exports.RevokeSecurityGroupEgressRequestFilterSensitiveLog = exports.RevokeClientVpnIngressResultFilterSensitiveLog = exports.RevokeClientVpnIngressRequestFilterSensitiveLog = exports.RestoreSnapshotTierResultFilterSensitiveLog = exports.RestoreSnapshotTierRequestFilterSensitiveLog = exports.RestoreSnapshotFromRecycleBinResultFilterSensitiveLog = exports.RestoreSnapshotFromRecycleBinRequestFilterSensitiveLog = exports.RestoreManagedPrefixListVersionResultFilterSensitiveLog = exports.RestoreManagedPrefixListVersionRequestFilterSensitiveLog = exports.RestoreImageFromRecycleBinResultFilterSensitiveLog = exports.RestoreImageFromRecycleBinRequestFilterSensitiveLog = exports.RestoreAddressToClassicResultFilterSensitiveLog = exports.RestoreAddressToClassicRequestFilterSensitiveLog = exports.ResetSnapshotAttributeRequestFilterSensitiveLog = exports.ResetNetworkInterfaceAttributeRequestFilterSensitiveLog = exports.ResetInstanceAttributeRequestFilterSensitiveLog = exports.ResetImageAttributeRequestFilterSensitiveLog = exports.ResetFpgaImageAttributeResultFilterSensitiveLog = exports.ResetFpgaImageAttributeRequestFilterSensitiveLog = exports.ResetEbsDefaultKmsKeyIdResultFilterSensitiveLog = exports.ResetEbsDefaultKmsKeyIdRequestFilterSensitiveLog = exports.ResetAddressAttributeResultFilterSensitiveLog = exports.ResetAddressAttributeRequestFilterSensitiveLog = exports.RequestSpotInstancesResultFilterSensitiveLog = exports.RequestSpotInstancesRequestFilterSensitiveLog = exports.RequestSpotLaunchSpecificationFilterSensitiveLog = exports.RequestSpotFleetResponseFilterSensitiveLog = exports.RequestSpotFleetRequestFilterSensitiveLog = exports.ReportInstanceStatusRequestFilterSensitiveLog = exports.ReplaceTransitGatewayRouteResultFilterSensitiveLog = exports.ReplaceTransitGatewayRouteRequestFilterSensitiveLog = exports.ReplaceRouteTableAssociationResultFilterSensitiveLog = exports.ReplaceRouteTableAssociationRequestFilterSensitiveLog = exports.ReplaceRouteRequestFilterSensitiveLog = exports.ReplaceNetworkAclEntryRequestFilterSensitiveLog = exports.ReplaceNetworkAclAssociationResultFilterSensitiveLog = exports.ReplaceNetworkAclAssociationRequestFilterSensitiveLog = exports.ReplaceIamInstanceProfileAssociationResultFilterSensitiveLog = exports.ReplaceIamInstanceProfileAssociationRequestFilterSensitiveLog = exports.ReleaseIpamPoolAllocationResultFilterSensitiveLog = exports.ReleaseIpamPoolAllocationRequestFilterSensitiveLog = exports.ReleaseHostsResultFilterSensitiveLog = exports.ReleaseHostsRequestFilterSensitiveLog = exports.ReleaseAddressRequestFilterSensitiveLog = exports.RejectVpcPeeringConnectionResultFilterSensitiveLog = exports.RejectVpcPeeringConnectionRequestFilterSensitiveLog = exports.RejectVpcEndpointConnectionsResultFilterSensitiveLog = exports.RejectVpcEndpointConnectionsRequestFilterSensitiveLog = exports.RejectTransitGatewayVpcAttachmentResultFilterSensitiveLog = exports.RejectTransitGatewayVpcAttachmentRequestFilterSensitiveLog = void 0;
exports.TerminateInstancesResultFilterSensitiveLog = exports.TerminateInstancesRequestFilterSensitiveLog = exports.TerminateClientVpnConnectionsResultFilterSensitiveLog = exports.TerminateConnectionStatusFilterSensitiveLog = exports.TerminateClientVpnConnectionsRequestFilterSensitiveLog = exports.StopInstancesResultFilterSensitiveLog = exports.StopInstancesRequestFilterSensitiveLog = exports.StartVpcEndpointServicePrivateDnsVerificationResultFilterSensitiveLog = exports.StartVpcEndpointServicePrivateDnsVerificationRequestFilterSensitiveLog = exports.StartNetworkInsightsAnalysisResultFilterSensitiveLog = exports.StartNetworkInsightsAnalysisRequestFilterSensitiveLog = exports.StartNetworkInsightsAccessScopeAnalysisResultFilterSensitiveLog = exports.StartNetworkInsightsAccessScopeAnalysisRequestFilterSensitiveLog = exports.StartInstancesResultFilterSensitiveLog = exports.InstanceStateChangeFilterSensitiveLog = exports.StartInstancesRequestFilterSensitiveLog = exports.SendDiagnosticInterruptRequestFilterSensitiveLog = exports.SearchTransitGatewayRoutesResultFilterSensitiveLog = exports.SearchTransitGatewayRoutesRequestFilterSensitiveLog = exports.SearchTransitGatewayMulticastGroupsResultFilterSensitiveLog = exports.TransitGatewayMulticastGroupFilterSensitiveLog = exports.SearchTransitGatewayMulticastGroupsRequestFilterSensitiveLog = exports.SearchLocalGatewayRoutesResultFilterSensitiveLog = exports.SearchLocalGatewayRoutesRequestFilterSensitiveLog = exports.RunScheduledInstancesResultFilterSensitiveLog = exports.RunScheduledInstancesRequestFilterSensitiveLog = exports.ScheduledInstancesLaunchSpecificationFilterSensitiveLog = exports.ScheduledInstancesPlacementFilterSensitiveLog = exports.ScheduledInstancesNetworkInterfaceFilterSensitiveLog = exports.ScheduledInstancesPrivateIpAddressConfigFilterSensitiveLog = exports.ScheduledInstancesIpv6AddressFilterSensitiveLog = exports.ScheduledInstancesMonitoringFilterSensitiveLog = exports.ScheduledInstancesIamInstanceProfileFilterSensitiveLog = exports.ScheduledInstancesBlockDeviceMappingFilterSensitiveLog = exports.ScheduledInstancesEbsFilterSensitiveLog = exports.RunInstancesRequestFilterSensitiveLog = exports.PrivateDnsNameOptionsRequestFilterSensitiveLog = exports.InstanceMetadataOptionsRequestFilterSensitiveLog = exports.InstanceMaintenanceOptionsRequestFilterSensitiveLog = exports.LicenseConfigurationRequestFilterSensitiveLog = exports.LaunchTemplateSpecificationFilterSensitiveLog = exports.InstanceMarketOptionsRequestFilterSensitiveLog = exports.SpotMarketOptionsFilterSensitiveLog = exports.HibernationOptionsRequestFilterSensitiveLog = exports.EnclaveOptionsRequestFilterSensitiveLog = exports.ElasticInferenceAcceleratorFilterSensitiveLog = exports.CpuOptionsRequestFilterSensitiveLog = exports.RevokeSecurityGroupIngressResultFilterSensitiveLog = exports.RevokeSecurityGroupIngressRequestFilterSensitiveLog = exports.RevokeSecurityGroupEgressResultFilterSensitiveLog = void 0;
const smithy_client_1 = require("@aws-sdk/smithy-client");
const models_4_1 = require("./models_4");
var ModifyAvailabilityZoneOptInStatus;
(function (ModifyAvailabilityZoneOptInStatus) {
    ModifyAvailabilityZoneOptInStatus["not_opted_in"] = "not-opted-in";
    ModifyAvailabilityZoneOptInStatus["opted_in"] = "opted-in";
})(ModifyAvailabilityZoneOptInStatus = exports.ModifyAvailabilityZoneOptInStatus || (exports.ModifyAvailabilityZoneOptInStatus = {}));
var OperationType;
(function (OperationType) {
    OperationType["add"] = "add";
    OperationType["remove"] = "remove";
})(OperationType = exports.OperationType || (exports.OperationType = {}));
var UnsuccessfulInstanceCreditSpecificationErrorCode;
(function (UnsuccessfulInstanceCreditSpecificationErrorCode) {
    UnsuccessfulInstanceCreditSpecificationErrorCode["INCORRECT_INSTANCE_STATE"] = "IncorrectInstanceState";
    UnsuccessfulInstanceCreditSpecificationErrorCode["INSTANCE_CREDIT_SPECIFICATION_NOT_SUPPORTED"] = "InstanceCreditSpecification.NotSupported";
    UnsuccessfulInstanceCreditSpecificationErrorCode["INSTANCE_NOT_FOUND"] = "InvalidInstanceID.NotFound";
    UnsuccessfulInstanceCreditSpecificationErrorCode["INVALID_INSTANCE_ID"] = "InvalidInstanceID.Malformed";
})(UnsuccessfulInstanceCreditSpecificationErrorCode = exports.UnsuccessfulInstanceCreditSpecificationErrorCode || (exports.UnsuccessfulInstanceCreditSpecificationErrorCode = {}));
var HostTenancy;
(function (HostTenancy) {
    HostTenancy["dedicated"] = "dedicated";
    HostTenancy["host"] = "host";
})(HostTenancy = exports.HostTenancy || (exports.HostTenancy = {}));
var TargetStorageTier;
(function (TargetStorageTier) {
    TargetStorageTier["archive"] = "archive";
})(TargetStorageTier = exports.TargetStorageTier || (exports.TargetStorageTier = {}));
var TrafficMirrorFilterRuleField;
(function (TrafficMirrorFilterRuleField) {
    TrafficMirrorFilterRuleField["description"] = "description";
    TrafficMirrorFilterRuleField["destination_port_range"] = "destination-port-range";
    TrafficMirrorFilterRuleField["protocol"] = "protocol";
    TrafficMirrorFilterRuleField["source_port_range"] = "source-port-range";
})(TrafficMirrorFilterRuleField = exports.TrafficMirrorFilterRuleField || (exports.TrafficMirrorFilterRuleField = {}));
var TrafficMirrorSessionField;
(function (TrafficMirrorSessionField) {
    TrafficMirrorSessionField["description"] = "description";
    TrafficMirrorSessionField["packet_length"] = "packet-length";
    TrafficMirrorSessionField["virtual_network_id"] = "virtual-network-id";
})(TrafficMirrorSessionField = exports.TrafficMirrorSessionField || (exports.TrafficMirrorSessionField = {}));
var VpcTenancy;
(function (VpcTenancy) {
    VpcTenancy["default"] = "default";
})(VpcTenancy = exports.VpcTenancy || (exports.VpcTenancy = {}));
var Status;
(function (Status) {
    Status["inClassic"] = "InClassic";
    Status["inVpc"] = "InVpc";
    Status["moveInProgress"] = "MoveInProgress";
})(Status = exports.Status || (exports.Status = {}));
var ReportInstanceReasonCodes;
(function (ReportInstanceReasonCodes) {
    ReportInstanceReasonCodes["instance_stuck_in_state"] = "instance-stuck-in-state";
    ReportInstanceReasonCodes["not_accepting_credentials"] = "not-accepting-credentials";
    ReportInstanceReasonCodes["other"] = "other";
    ReportInstanceReasonCodes["password_not_available"] = "password-not-available";
    ReportInstanceReasonCodes["performance_ebs_volume"] = "performance-ebs-volume";
    ReportInstanceReasonCodes["performance_instance_store"] = "performance-instance-store";
    ReportInstanceReasonCodes["performance_network"] = "performance-network";
    ReportInstanceReasonCodes["performance_other"] = "performance-other";
    ReportInstanceReasonCodes["unresponsive"] = "unresponsive";
})(ReportInstanceReasonCodes = exports.ReportInstanceReasonCodes || (exports.ReportInstanceReasonCodes = {}));
var ReportStatusType;
(function (ReportStatusType) {
    ReportStatusType["impaired"] = "impaired";
    ReportStatusType["ok"] = "ok";
})(ReportStatusType = exports.ReportStatusType || (exports.ReportStatusType = {}));
var ResetFpgaImageAttributeName;
(function (ResetFpgaImageAttributeName) {
    ResetFpgaImageAttributeName["loadPermission"] = "loadPermission";
})(ResetFpgaImageAttributeName = exports.ResetFpgaImageAttributeName || (exports.ResetFpgaImageAttributeName = {}));
var ResetImageAttributeName;
(function (ResetImageAttributeName) {
    ResetImageAttributeName["launchPermission"] = "launchPermission";
})(ResetImageAttributeName = exports.ResetImageAttributeName || (exports.ResetImageAttributeName = {}));
var MembershipType;
(function (MembershipType) {
    MembershipType["igmp"] = "igmp";
    MembershipType["static"] = "static";
})(MembershipType = exports.MembershipType || (exports.MembershipType = {}));
const ModifyAvailabilityZoneGroupRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyAvailabilityZoneGroupRequestFilterSensitiveLog = ModifyAvailabilityZoneGroupRequestFilterSensitiveLog;
const ModifyAvailabilityZoneGroupResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyAvailabilityZoneGroupResultFilterSensitiveLog = ModifyAvailabilityZoneGroupResultFilterSensitiveLog;
const ModifyCapacityReservationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyCapacityReservationRequestFilterSensitiveLog = ModifyCapacityReservationRequestFilterSensitiveLog;
const ModifyCapacityReservationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyCapacityReservationResultFilterSensitiveLog = ModifyCapacityReservationResultFilterSensitiveLog;
const ModifyCapacityReservationFleetRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyCapacityReservationFleetRequestFilterSensitiveLog = ModifyCapacityReservationFleetRequestFilterSensitiveLog;
const ModifyCapacityReservationFleetResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyCapacityReservationFleetResultFilterSensitiveLog = ModifyCapacityReservationFleetResultFilterSensitiveLog;
const DnsServersOptionsModifyStructureFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.DnsServersOptionsModifyStructureFilterSensitiveLog = DnsServersOptionsModifyStructureFilterSensitiveLog;
const ModifyClientVpnEndpointRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyClientVpnEndpointRequestFilterSensitiveLog = ModifyClientVpnEndpointRequestFilterSensitiveLog;
const ModifyClientVpnEndpointResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyClientVpnEndpointResultFilterSensitiveLog = ModifyClientVpnEndpointResultFilterSensitiveLog;
const ModifyDefaultCreditSpecificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyDefaultCreditSpecificationRequestFilterSensitiveLog = ModifyDefaultCreditSpecificationRequestFilterSensitiveLog;
const ModifyDefaultCreditSpecificationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyDefaultCreditSpecificationResultFilterSensitiveLog = ModifyDefaultCreditSpecificationResultFilterSensitiveLog;
const ModifyEbsDefaultKmsKeyIdRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyEbsDefaultKmsKeyIdRequestFilterSensitiveLog = ModifyEbsDefaultKmsKeyIdRequestFilterSensitiveLog;
const ModifyEbsDefaultKmsKeyIdResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyEbsDefaultKmsKeyIdResultFilterSensitiveLog = ModifyEbsDefaultKmsKeyIdResultFilterSensitiveLog;
const ModifyFleetRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyFleetRequestFilterSensitiveLog = ModifyFleetRequestFilterSensitiveLog;
const ModifyFleetResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyFleetResultFilterSensitiveLog = ModifyFleetResultFilterSensitiveLog;
const LoadPermissionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.LoadPermissionRequestFilterSensitiveLog = LoadPermissionRequestFilterSensitiveLog;
const LoadPermissionModificationsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.LoadPermissionModificationsFilterSensitiveLog = LoadPermissionModificationsFilterSensitiveLog;
const ModifyFpgaImageAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyFpgaImageAttributeRequestFilterSensitiveLog = ModifyFpgaImageAttributeRequestFilterSensitiveLog;
const ModifyFpgaImageAttributeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyFpgaImageAttributeResultFilterSensitiveLog = ModifyFpgaImageAttributeResultFilterSensitiveLog;
const ModifyHostsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyHostsRequestFilterSensitiveLog = ModifyHostsRequestFilterSensitiveLog;
const ModifyHostsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyHostsResultFilterSensitiveLog = ModifyHostsResultFilterSensitiveLog;
const ModifyIdentityIdFormatRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIdentityIdFormatRequestFilterSensitiveLog = ModifyIdentityIdFormatRequestFilterSensitiveLog;
const ModifyIdFormatRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIdFormatRequestFilterSensitiveLog = ModifyIdFormatRequestFilterSensitiveLog;
const LaunchPermissionModificationsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.LaunchPermissionModificationsFilterSensitiveLog = LaunchPermissionModificationsFilterSensitiveLog;
const ModifyImageAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyImageAttributeRequestFilterSensitiveLog = ModifyImageAttributeRequestFilterSensitiveLog;
const EbsInstanceBlockDeviceSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EbsInstanceBlockDeviceSpecificationFilterSensitiveLog = EbsInstanceBlockDeviceSpecificationFilterSensitiveLog;
const InstanceBlockDeviceMappingSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceBlockDeviceMappingSpecificationFilterSensitiveLog = InstanceBlockDeviceMappingSpecificationFilterSensitiveLog;
const BlobAttributeValueFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.BlobAttributeValueFilterSensitiveLog = BlobAttributeValueFilterSensitiveLog;
const ModifyInstanceAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceAttributeRequestFilterSensitiveLog = ModifyInstanceAttributeRequestFilterSensitiveLog;
const CapacityReservationSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.CapacityReservationSpecificationFilterSensitiveLog = CapacityReservationSpecificationFilterSensitiveLog;
const ModifyInstanceCapacityReservationAttributesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceCapacityReservationAttributesRequestFilterSensitiveLog = ModifyInstanceCapacityReservationAttributesRequestFilterSensitiveLog;
const ModifyInstanceCapacityReservationAttributesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceCapacityReservationAttributesResultFilterSensitiveLog = ModifyInstanceCapacityReservationAttributesResultFilterSensitiveLog;
const InstanceCreditSpecificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceCreditSpecificationRequestFilterSensitiveLog = InstanceCreditSpecificationRequestFilterSensitiveLog;
const ModifyInstanceCreditSpecificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceCreditSpecificationRequestFilterSensitiveLog = ModifyInstanceCreditSpecificationRequestFilterSensitiveLog;
const SuccessfulInstanceCreditSpecificationItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SuccessfulInstanceCreditSpecificationItemFilterSensitiveLog = SuccessfulInstanceCreditSpecificationItemFilterSensitiveLog;
const UnsuccessfulInstanceCreditSpecificationItemErrorFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.UnsuccessfulInstanceCreditSpecificationItemErrorFilterSensitiveLog = UnsuccessfulInstanceCreditSpecificationItemErrorFilterSensitiveLog;
const UnsuccessfulInstanceCreditSpecificationItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.UnsuccessfulInstanceCreditSpecificationItemFilterSensitiveLog = UnsuccessfulInstanceCreditSpecificationItemFilterSensitiveLog;
const ModifyInstanceCreditSpecificationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceCreditSpecificationResultFilterSensitiveLog = ModifyInstanceCreditSpecificationResultFilterSensitiveLog;
const ModifyInstanceEventStartTimeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceEventStartTimeRequestFilterSensitiveLog = ModifyInstanceEventStartTimeRequestFilterSensitiveLog;
const ModifyInstanceEventStartTimeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceEventStartTimeResultFilterSensitiveLog = ModifyInstanceEventStartTimeResultFilterSensitiveLog;
const ModifyInstanceEventWindowRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceEventWindowRequestFilterSensitiveLog = ModifyInstanceEventWindowRequestFilterSensitiveLog;
const ModifyInstanceEventWindowResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceEventWindowResultFilterSensitiveLog = ModifyInstanceEventWindowResultFilterSensitiveLog;
const ModifyInstanceMaintenanceOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceMaintenanceOptionsRequestFilterSensitiveLog = ModifyInstanceMaintenanceOptionsRequestFilterSensitiveLog;
const ModifyInstanceMaintenanceOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceMaintenanceOptionsResultFilterSensitiveLog = ModifyInstanceMaintenanceOptionsResultFilterSensitiveLog;
const ModifyInstanceMetadataOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceMetadataOptionsRequestFilterSensitiveLog = ModifyInstanceMetadataOptionsRequestFilterSensitiveLog;
const ModifyInstanceMetadataOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstanceMetadataOptionsResultFilterSensitiveLog = ModifyInstanceMetadataOptionsResultFilterSensitiveLog;
const ModifyInstancePlacementRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstancePlacementRequestFilterSensitiveLog = ModifyInstancePlacementRequestFilterSensitiveLog;
const ModifyInstancePlacementResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyInstancePlacementResultFilterSensitiveLog = ModifyInstancePlacementResultFilterSensitiveLog;
const RemoveIpamOperatingRegionFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RemoveIpamOperatingRegionFilterSensitiveLog = RemoveIpamOperatingRegionFilterSensitiveLog;
const ModifyIpamRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamRequestFilterSensitiveLog = ModifyIpamRequestFilterSensitiveLog;
const ModifyIpamResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamResultFilterSensitiveLog = ModifyIpamResultFilterSensitiveLog;
const ModifyIpamPoolRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamPoolRequestFilterSensitiveLog = ModifyIpamPoolRequestFilterSensitiveLog;
const ModifyIpamPoolResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamPoolResultFilterSensitiveLog = ModifyIpamPoolResultFilterSensitiveLog;
const ModifyIpamResourceCidrRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamResourceCidrRequestFilterSensitiveLog = ModifyIpamResourceCidrRequestFilterSensitiveLog;
const ModifyIpamResourceCidrResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamResourceCidrResultFilterSensitiveLog = ModifyIpamResourceCidrResultFilterSensitiveLog;
const ModifyIpamResourceDiscoveryRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamResourceDiscoveryRequestFilterSensitiveLog = ModifyIpamResourceDiscoveryRequestFilterSensitiveLog;
const ModifyIpamResourceDiscoveryResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamResourceDiscoveryResultFilterSensitiveLog = ModifyIpamResourceDiscoveryResultFilterSensitiveLog;
const ModifyIpamScopeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamScopeRequestFilterSensitiveLog = ModifyIpamScopeRequestFilterSensitiveLog;
const ModifyIpamScopeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyIpamScopeResultFilterSensitiveLog = ModifyIpamScopeResultFilterSensitiveLog;
const ModifyLaunchTemplateRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyLaunchTemplateRequestFilterSensitiveLog = ModifyLaunchTemplateRequestFilterSensitiveLog;
const ModifyLaunchTemplateResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyLaunchTemplateResultFilterSensitiveLog = ModifyLaunchTemplateResultFilterSensitiveLog;
const ModifyLocalGatewayRouteRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyLocalGatewayRouteRequestFilterSensitiveLog = ModifyLocalGatewayRouteRequestFilterSensitiveLog;
const ModifyLocalGatewayRouteResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyLocalGatewayRouteResultFilterSensitiveLog = ModifyLocalGatewayRouteResultFilterSensitiveLog;
const RemovePrefixListEntryFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RemovePrefixListEntryFilterSensitiveLog = RemovePrefixListEntryFilterSensitiveLog;
const ModifyManagedPrefixListRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyManagedPrefixListRequestFilterSensitiveLog = ModifyManagedPrefixListRequestFilterSensitiveLog;
const ModifyManagedPrefixListResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyManagedPrefixListResultFilterSensitiveLog = ModifyManagedPrefixListResultFilterSensitiveLog;
const NetworkInterfaceAttachmentChangesFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.NetworkInterfaceAttachmentChangesFilterSensitiveLog = NetworkInterfaceAttachmentChangesFilterSensitiveLog;
const ModifyNetworkInterfaceAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyNetworkInterfaceAttributeRequestFilterSensitiveLog = ModifyNetworkInterfaceAttributeRequestFilterSensitiveLog;
const ModifyPrivateDnsNameOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyPrivateDnsNameOptionsRequestFilterSensitiveLog = ModifyPrivateDnsNameOptionsRequestFilterSensitiveLog;
const ModifyPrivateDnsNameOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyPrivateDnsNameOptionsResultFilterSensitiveLog = ModifyPrivateDnsNameOptionsResultFilterSensitiveLog;
const ModifyReservedInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyReservedInstancesRequestFilterSensitiveLog = ModifyReservedInstancesRequestFilterSensitiveLog;
const ModifyReservedInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyReservedInstancesResultFilterSensitiveLog = ModifyReservedInstancesResultFilterSensitiveLog;
const SecurityGroupRuleRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SecurityGroupRuleRequestFilterSensitiveLog = SecurityGroupRuleRequestFilterSensitiveLog;
const SecurityGroupRuleUpdateFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SecurityGroupRuleUpdateFilterSensitiveLog = SecurityGroupRuleUpdateFilterSensitiveLog;
const ModifySecurityGroupRulesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifySecurityGroupRulesRequestFilterSensitiveLog = ModifySecurityGroupRulesRequestFilterSensitiveLog;
const ModifySecurityGroupRulesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifySecurityGroupRulesResultFilterSensitiveLog = ModifySecurityGroupRulesResultFilterSensitiveLog;
const CreateVolumePermissionModificationsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.CreateVolumePermissionModificationsFilterSensitiveLog = CreateVolumePermissionModificationsFilterSensitiveLog;
const ModifySnapshotAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifySnapshotAttributeRequestFilterSensitiveLog = ModifySnapshotAttributeRequestFilterSensitiveLog;
const ModifySnapshotTierRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifySnapshotTierRequestFilterSensitiveLog = ModifySnapshotTierRequestFilterSensitiveLog;
const ModifySnapshotTierResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifySnapshotTierResultFilterSensitiveLog = ModifySnapshotTierResultFilterSensitiveLog;
const ModifySpotFleetRequestRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifySpotFleetRequestRequestFilterSensitiveLog = ModifySpotFleetRequestRequestFilterSensitiveLog;
const ModifySpotFleetRequestResponseFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifySpotFleetRequestResponseFilterSensitiveLog = ModifySpotFleetRequestResponseFilterSensitiveLog;
const ModifySubnetAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifySubnetAttributeRequestFilterSensitiveLog = ModifySubnetAttributeRequestFilterSensitiveLog;
const ModifyTrafficMirrorFilterNetworkServicesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTrafficMirrorFilterNetworkServicesRequestFilterSensitiveLog = ModifyTrafficMirrorFilterNetworkServicesRequestFilterSensitiveLog;
const ModifyTrafficMirrorFilterNetworkServicesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTrafficMirrorFilterNetworkServicesResultFilterSensitiveLog = ModifyTrafficMirrorFilterNetworkServicesResultFilterSensitiveLog;
const ModifyTrafficMirrorFilterRuleRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTrafficMirrorFilterRuleRequestFilterSensitiveLog = ModifyTrafficMirrorFilterRuleRequestFilterSensitiveLog;
const ModifyTrafficMirrorFilterRuleResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTrafficMirrorFilterRuleResultFilterSensitiveLog = ModifyTrafficMirrorFilterRuleResultFilterSensitiveLog;
const ModifyTrafficMirrorSessionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTrafficMirrorSessionRequestFilterSensitiveLog = ModifyTrafficMirrorSessionRequestFilterSensitiveLog;
const ModifyTrafficMirrorSessionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTrafficMirrorSessionResultFilterSensitiveLog = ModifyTrafficMirrorSessionResultFilterSensitiveLog;
const ModifyTransitGatewayOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTransitGatewayOptionsFilterSensitiveLog = ModifyTransitGatewayOptionsFilterSensitiveLog;
const ModifyTransitGatewayRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTransitGatewayRequestFilterSensitiveLog = ModifyTransitGatewayRequestFilterSensitiveLog;
const ModifyTransitGatewayResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTransitGatewayResultFilterSensitiveLog = ModifyTransitGatewayResultFilterSensitiveLog;
const ModifyTransitGatewayPrefixListReferenceRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTransitGatewayPrefixListReferenceRequestFilterSensitiveLog = ModifyTransitGatewayPrefixListReferenceRequestFilterSensitiveLog;
const ModifyTransitGatewayPrefixListReferenceResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTransitGatewayPrefixListReferenceResultFilterSensitiveLog = ModifyTransitGatewayPrefixListReferenceResultFilterSensitiveLog;
const ModifyTransitGatewayVpcAttachmentRequestOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTransitGatewayVpcAttachmentRequestOptionsFilterSensitiveLog = ModifyTransitGatewayVpcAttachmentRequestOptionsFilterSensitiveLog;
const ModifyTransitGatewayVpcAttachmentRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTransitGatewayVpcAttachmentRequestFilterSensitiveLog = ModifyTransitGatewayVpcAttachmentRequestFilterSensitiveLog;
const ModifyTransitGatewayVpcAttachmentResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyTransitGatewayVpcAttachmentResultFilterSensitiveLog = ModifyTransitGatewayVpcAttachmentResultFilterSensitiveLog;
const ModifyVerifiedAccessEndpointLoadBalancerOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessEndpointLoadBalancerOptionsFilterSensitiveLog = ModifyVerifiedAccessEndpointLoadBalancerOptionsFilterSensitiveLog;
const ModifyVerifiedAccessEndpointEniOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessEndpointEniOptionsFilterSensitiveLog = ModifyVerifiedAccessEndpointEniOptionsFilterSensitiveLog;
const ModifyVerifiedAccessEndpointRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessEndpointRequestFilterSensitiveLog = ModifyVerifiedAccessEndpointRequestFilterSensitiveLog;
const ModifyVerifiedAccessEndpointResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessEndpointResultFilterSensitiveLog = ModifyVerifiedAccessEndpointResultFilterSensitiveLog;
const ModifyVerifiedAccessEndpointPolicyRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessEndpointPolicyRequestFilterSensitiveLog = ModifyVerifiedAccessEndpointPolicyRequestFilterSensitiveLog;
const ModifyVerifiedAccessEndpointPolicyResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessEndpointPolicyResultFilterSensitiveLog = ModifyVerifiedAccessEndpointPolicyResultFilterSensitiveLog;
const ModifyVerifiedAccessGroupRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessGroupRequestFilterSensitiveLog = ModifyVerifiedAccessGroupRequestFilterSensitiveLog;
const ModifyVerifiedAccessGroupResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessGroupResultFilterSensitiveLog = ModifyVerifiedAccessGroupResultFilterSensitiveLog;
const ModifyVerifiedAccessGroupPolicyRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessGroupPolicyRequestFilterSensitiveLog = ModifyVerifiedAccessGroupPolicyRequestFilterSensitiveLog;
const ModifyVerifiedAccessGroupPolicyResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessGroupPolicyResultFilterSensitiveLog = ModifyVerifiedAccessGroupPolicyResultFilterSensitiveLog;
const ModifyVerifiedAccessInstanceRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessInstanceRequestFilterSensitiveLog = ModifyVerifiedAccessInstanceRequestFilterSensitiveLog;
const ModifyVerifiedAccessInstanceResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessInstanceResultFilterSensitiveLog = ModifyVerifiedAccessInstanceResultFilterSensitiveLog;
const VerifiedAccessLogCloudWatchLogsDestinationOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VerifiedAccessLogCloudWatchLogsDestinationOptionsFilterSensitiveLog = VerifiedAccessLogCloudWatchLogsDestinationOptionsFilterSensitiveLog;
const VerifiedAccessLogKinesisDataFirehoseDestinationOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VerifiedAccessLogKinesisDataFirehoseDestinationOptionsFilterSensitiveLog = VerifiedAccessLogKinesisDataFirehoseDestinationOptionsFilterSensitiveLog;
const VerifiedAccessLogS3DestinationOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VerifiedAccessLogS3DestinationOptionsFilterSensitiveLog = VerifiedAccessLogS3DestinationOptionsFilterSensitiveLog;
const VerifiedAccessLogOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.VerifiedAccessLogOptionsFilterSensitiveLog = VerifiedAccessLogOptionsFilterSensitiveLog;
const ModifyVerifiedAccessInstanceLoggingConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessInstanceLoggingConfigurationRequestFilterSensitiveLog = ModifyVerifiedAccessInstanceLoggingConfigurationRequestFilterSensitiveLog;
const ModifyVerifiedAccessInstanceLoggingConfigurationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessInstanceLoggingConfigurationResultFilterSensitiveLog = ModifyVerifiedAccessInstanceLoggingConfigurationResultFilterSensitiveLog;
const ModifyVerifiedAccessTrustProviderOidcOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessTrustProviderOidcOptionsFilterSensitiveLog = ModifyVerifiedAccessTrustProviderOidcOptionsFilterSensitiveLog;
const ModifyVerifiedAccessTrustProviderRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessTrustProviderRequestFilterSensitiveLog = ModifyVerifiedAccessTrustProviderRequestFilterSensitiveLog;
const ModifyVerifiedAccessTrustProviderResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVerifiedAccessTrustProviderResultFilterSensitiveLog = ModifyVerifiedAccessTrustProviderResultFilterSensitiveLog;
const ModifyVolumeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVolumeRequestFilterSensitiveLog = ModifyVolumeRequestFilterSensitiveLog;
const ModifyVolumeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVolumeResultFilterSensitiveLog = ModifyVolumeResultFilterSensitiveLog;
const ModifyVolumeAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVolumeAttributeRequestFilterSensitiveLog = ModifyVolumeAttributeRequestFilterSensitiveLog;
const ModifyVpcAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcAttributeRequestFilterSensitiveLog = ModifyVpcAttributeRequestFilterSensitiveLog;
const ModifyVpcEndpointRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointRequestFilterSensitiveLog = ModifyVpcEndpointRequestFilterSensitiveLog;
const ModifyVpcEndpointResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointResultFilterSensitiveLog = ModifyVpcEndpointResultFilterSensitiveLog;
const ModifyVpcEndpointConnectionNotificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointConnectionNotificationRequestFilterSensitiveLog = ModifyVpcEndpointConnectionNotificationRequestFilterSensitiveLog;
const ModifyVpcEndpointConnectionNotificationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointConnectionNotificationResultFilterSensitiveLog = ModifyVpcEndpointConnectionNotificationResultFilterSensitiveLog;
const ModifyVpcEndpointServiceConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointServiceConfigurationRequestFilterSensitiveLog = ModifyVpcEndpointServiceConfigurationRequestFilterSensitiveLog;
const ModifyVpcEndpointServiceConfigurationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointServiceConfigurationResultFilterSensitiveLog = ModifyVpcEndpointServiceConfigurationResultFilterSensitiveLog;
const ModifyVpcEndpointServicePayerResponsibilityRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointServicePayerResponsibilityRequestFilterSensitiveLog = ModifyVpcEndpointServicePayerResponsibilityRequestFilterSensitiveLog;
const ModifyVpcEndpointServicePayerResponsibilityResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointServicePayerResponsibilityResultFilterSensitiveLog = ModifyVpcEndpointServicePayerResponsibilityResultFilterSensitiveLog;
const ModifyVpcEndpointServicePermissionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointServicePermissionsRequestFilterSensitiveLog = ModifyVpcEndpointServicePermissionsRequestFilterSensitiveLog;
const ModifyVpcEndpointServicePermissionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcEndpointServicePermissionsResultFilterSensitiveLog = ModifyVpcEndpointServicePermissionsResultFilterSensitiveLog;
const PeeringConnectionOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PeeringConnectionOptionsRequestFilterSensitiveLog = PeeringConnectionOptionsRequestFilterSensitiveLog;
const ModifyVpcPeeringConnectionOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcPeeringConnectionOptionsRequestFilterSensitiveLog = ModifyVpcPeeringConnectionOptionsRequestFilterSensitiveLog;
const PeeringConnectionOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PeeringConnectionOptionsFilterSensitiveLog = PeeringConnectionOptionsFilterSensitiveLog;
const ModifyVpcPeeringConnectionOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcPeeringConnectionOptionsResultFilterSensitiveLog = ModifyVpcPeeringConnectionOptionsResultFilterSensitiveLog;
const ModifyVpcTenancyRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcTenancyRequestFilterSensitiveLog = ModifyVpcTenancyRequestFilterSensitiveLog;
const ModifyVpcTenancyResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpcTenancyResultFilterSensitiveLog = ModifyVpcTenancyResultFilterSensitiveLog;
const ModifyVpnConnectionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpnConnectionRequestFilterSensitiveLog = ModifyVpnConnectionRequestFilterSensitiveLog;
const ModifyVpnConnectionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpnConnectionResultFilterSensitiveLog = ModifyVpnConnectionResultFilterSensitiveLog;
const ModifyVpnConnectionOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpnConnectionOptionsRequestFilterSensitiveLog = ModifyVpnConnectionOptionsRequestFilterSensitiveLog;
const ModifyVpnConnectionOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpnConnectionOptionsResultFilterSensitiveLog = ModifyVpnConnectionOptionsResultFilterSensitiveLog;
const ModifyVpnTunnelCertificateRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpnTunnelCertificateRequestFilterSensitiveLog = ModifyVpnTunnelCertificateRequestFilterSensitiveLog;
const ModifyVpnTunnelCertificateResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpnTunnelCertificateResultFilterSensitiveLog = ModifyVpnTunnelCertificateResultFilterSensitiveLog;
const ModifyVpnTunnelOptionsSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpnTunnelOptionsSpecificationFilterSensitiveLog = ModifyVpnTunnelOptionsSpecificationFilterSensitiveLog;
const ModifyVpnTunnelOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpnTunnelOptionsRequestFilterSensitiveLog = ModifyVpnTunnelOptionsRequestFilterSensitiveLog;
const ModifyVpnTunnelOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ModifyVpnTunnelOptionsResultFilterSensitiveLog = ModifyVpnTunnelOptionsResultFilterSensitiveLog;
const MonitorInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.MonitorInstancesRequestFilterSensitiveLog = MonitorInstancesRequestFilterSensitiveLog;
const InstanceMonitoringFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceMonitoringFilterSensitiveLog = InstanceMonitoringFilterSensitiveLog;
const MonitorInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.MonitorInstancesResultFilterSensitiveLog = MonitorInstancesResultFilterSensitiveLog;
const MoveAddressToVpcRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.MoveAddressToVpcRequestFilterSensitiveLog = MoveAddressToVpcRequestFilterSensitiveLog;
const MoveAddressToVpcResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.MoveAddressToVpcResultFilterSensitiveLog = MoveAddressToVpcResultFilterSensitiveLog;
const MoveByoipCidrToIpamRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.MoveByoipCidrToIpamRequestFilterSensitiveLog = MoveByoipCidrToIpamRequestFilterSensitiveLog;
const MoveByoipCidrToIpamResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.MoveByoipCidrToIpamResultFilterSensitiveLog = MoveByoipCidrToIpamResultFilterSensitiveLog;
const CidrAuthorizationContextFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.CidrAuthorizationContextFilterSensitiveLog = CidrAuthorizationContextFilterSensitiveLog;
const ProvisionByoipCidrRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ProvisionByoipCidrRequestFilterSensitiveLog = ProvisionByoipCidrRequestFilterSensitiveLog;
const ProvisionByoipCidrResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ProvisionByoipCidrResultFilterSensitiveLog = ProvisionByoipCidrResultFilterSensitiveLog;
const IpamCidrAuthorizationContextFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.IpamCidrAuthorizationContextFilterSensitiveLog = IpamCidrAuthorizationContextFilterSensitiveLog;
const ProvisionIpamPoolCidrRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ProvisionIpamPoolCidrRequestFilterSensitiveLog = ProvisionIpamPoolCidrRequestFilterSensitiveLog;
const ProvisionIpamPoolCidrResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ProvisionIpamPoolCidrResultFilterSensitiveLog = ProvisionIpamPoolCidrResultFilterSensitiveLog;
const ProvisionPublicIpv4PoolCidrRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ProvisionPublicIpv4PoolCidrRequestFilterSensitiveLog = ProvisionPublicIpv4PoolCidrRequestFilterSensitiveLog;
const ProvisionPublicIpv4PoolCidrResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ProvisionPublicIpv4PoolCidrResultFilterSensitiveLog = ProvisionPublicIpv4PoolCidrResultFilterSensitiveLog;
const PurchaseHostReservationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PurchaseHostReservationRequestFilterSensitiveLog = PurchaseHostReservationRequestFilterSensitiveLog;
const PurchaseHostReservationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PurchaseHostReservationResultFilterSensitiveLog = PurchaseHostReservationResultFilterSensitiveLog;
const ReservedInstanceLimitPriceFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReservedInstanceLimitPriceFilterSensitiveLog = ReservedInstanceLimitPriceFilterSensitiveLog;
const PurchaseReservedInstancesOfferingRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PurchaseReservedInstancesOfferingRequestFilterSensitiveLog = PurchaseReservedInstancesOfferingRequestFilterSensitiveLog;
const PurchaseReservedInstancesOfferingResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PurchaseReservedInstancesOfferingResultFilterSensitiveLog = PurchaseReservedInstancesOfferingResultFilterSensitiveLog;
const PurchaseRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PurchaseRequestFilterSensitiveLog = PurchaseRequestFilterSensitiveLog;
const PurchaseScheduledInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PurchaseScheduledInstancesRequestFilterSensitiveLog = PurchaseScheduledInstancesRequestFilterSensitiveLog;
const PurchaseScheduledInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PurchaseScheduledInstancesResultFilterSensitiveLog = PurchaseScheduledInstancesResultFilterSensitiveLog;
const RebootInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RebootInstancesRequestFilterSensitiveLog = RebootInstancesRequestFilterSensitiveLog;
const RegisterImageRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RegisterImageRequestFilterSensitiveLog = RegisterImageRequestFilterSensitiveLog;
const RegisterImageResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RegisterImageResultFilterSensitiveLog = RegisterImageResultFilterSensitiveLog;
const RegisterInstanceTagAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RegisterInstanceTagAttributeRequestFilterSensitiveLog = RegisterInstanceTagAttributeRequestFilterSensitiveLog;
const RegisterInstanceEventNotificationAttributesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RegisterInstanceEventNotificationAttributesRequestFilterSensitiveLog = RegisterInstanceEventNotificationAttributesRequestFilterSensitiveLog;
const RegisterInstanceEventNotificationAttributesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RegisterInstanceEventNotificationAttributesResultFilterSensitiveLog = RegisterInstanceEventNotificationAttributesResultFilterSensitiveLog;
const RegisterTransitGatewayMulticastGroupMembersRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RegisterTransitGatewayMulticastGroupMembersRequestFilterSensitiveLog = RegisterTransitGatewayMulticastGroupMembersRequestFilterSensitiveLog;
const TransitGatewayMulticastRegisteredGroupMembersFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayMulticastRegisteredGroupMembersFilterSensitiveLog = TransitGatewayMulticastRegisteredGroupMembersFilterSensitiveLog;
const RegisterTransitGatewayMulticastGroupMembersResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RegisterTransitGatewayMulticastGroupMembersResultFilterSensitiveLog = RegisterTransitGatewayMulticastGroupMembersResultFilterSensitiveLog;
const RegisterTransitGatewayMulticastGroupSourcesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RegisterTransitGatewayMulticastGroupSourcesRequestFilterSensitiveLog = RegisterTransitGatewayMulticastGroupSourcesRequestFilterSensitiveLog;
const TransitGatewayMulticastRegisteredGroupSourcesFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayMulticastRegisteredGroupSourcesFilterSensitiveLog = TransitGatewayMulticastRegisteredGroupSourcesFilterSensitiveLog;
const RegisterTransitGatewayMulticastGroupSourcesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RegisterTransitGatewayMulticastGroupSourcesResultFilterSensitiveLog = RegisterTransitGatewayMulticastGroupSourcesResultFilterSensitiveLog;
const RejectTransitGatewayMulticastDomainAssociationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectTransitGatewayMulticastDomainAssociationsRequestFilterSensitiveLog = RejectTransitGatewayMulticastDomainAssociationsRequestFilterSensitiveLog;
const RejectTransitGatewayMulticastDomainAssociationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectTransitGatewayMulticastDomainAssociationsResultFilterSensitiveLog = RejectTransitGatewayMulticastDomainAssociationsResultFilterSensitiveLog;
const RejectTransitGatewayPeeringAttachmentRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectTransitGatewayPeeringAttachmentRequestFilterSensitiveLog = RejectTransitGatewayPeeringAttachmentRequestFilterSensitiveLog;
const RejectTransitGatewayPeeringAttachmentResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectTransitGatewayPeeringAttachmentResultFilterSensitiveLog = RejectTransitGatewayPeeringAttachmentResultFilterSensitiveLog;
const RejectTransitGatewayVpcAttachmentRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectTransitGatewayVpcAttachmentRequestFilterSensitiveLog = RejectTransitGatewayVpcAttachmentRequestFilterSensitiveLog;
const RejectTransitGatewayVpcAttachmentResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectTransitGatewayVpcAttachmentResultFilterSensitiveLog = RejectTransitGatewayVpcAttachmentResultFilterSensitiveLog;
const RejectVpcEndpointConnectionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectVpcEndpointConnectionsRequestFilterSensitiveLog = RejectVpcEndpointConnectionsRequestFilterSensitiveLog;
const RejectVpcEndpointConnectionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectVpcEndpointConnectionsResultFilterSensitiveLog = RejectVpcEndpointConnectionsResultFilterSensitiveLog;
const RejectVpcPeeringConnectionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectVpcPeeringConnectionRequestFilterSensitiveLog = RejectVpcPeeringConnectionRequestFilterSensitiveLog;
const RejectVpcPeeringConnectionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RejectVpcPeeringConnectionResultFilterSensitiveLog = RejectVpcPeeringConnectionResultFilterSensitiveLog;
const ReleaseAddressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReleaseAddressRequestFilterSensitiveLog = ReleaseAddressRequestFilterSensitiveLog;
const ReleaseHostsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReleaseHostsRequestFilterSensitiveLog = ReleaseHostsRequestFilterSensitiveLog;
const ReleaseHostsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReleaseHostsResultFilterSensitiveLog = ReleaseHostsResultFilterSensitiveLog;
const ReleaseIpamPoolAllocationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReleaseIpamPoolAllocationRequestFilterSensitiveLog = ReleaseIpamPoolAllocationRequestFilterSensitiveLog;
const ReleaseIpamPoolAllocationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReleaseIpamPoolAllocationResultFilterSensitiveLog = ReleaseIpamPoolAllocationResultFilterSensitiveLog;
const ReplaceIamInstanceProfileAssociationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceIamInstanceProfileAssociationRequestFilterSensitiveLog = ReplaceIamInstanceProfileAssociationRequestFilterSensitiveLog;
const ReplaceIamInstanceProfileAssociationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceIamInstanceProfileAssociationResultFilterSensitiveLog = ReplaceIamInstanceProfileAssociationResultFilterSensitiveLog;
const ReplaceNetworkAclAssociationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceNetworkAclAssociationRequestFilterSensitiveLog = ReplaceNetworkAclAssociationRequestFilterSensitiveLog;
const ReplaceNetworkAclAssociationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceNetworkAclAssociationResultFilterSensitiveLog = ReplaceNetworkAclAssociationResultFilterSensitiveLog;
const ReplaceNetworkAclEntryRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceNetworkAclEntryRequestFilterSensitiveLog = ReplaceNetworkAclEntryRequestFilterSensitiveLog;
const ReplaceRouteRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceRouteRequestFilterSensitiveLog = ReplaceRouteRequestFilterSensitiveLog;
const ReplaceRouteTableAssociationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceRouteTableAssociationRequestFilterSensitiveLog = ReplaceRouteTableAssociationRequestFilterSensitiveLog;
const ReplaceRouteTableAssociationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceRouteTableAssociationResultFilterSensitiveLog = ReplaceRouteTableAssociationResultFilterSensitiveLog;
const ReplaceTransitGatewayRouteRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceTransitGatewayRouteRequestFilterSensitiveLog = ReplaceTransitGatewayRouteRequestFilterSensitiveLog;
const ReplaceTransitGatewayRouteResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReplaceTransitGatewayRouteResultFilterSensitiveLog = ReplaceTransitGatewayRouteResultFilterSensitiveLog;
const ReportInstanceStatusRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ReportInstanceStatusRequestFilterSensitiveLog = ReportInstanceStatusRequestFilterSensitiveLog;
const RequestSpotFleetRequestFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.SpotFleetRequestConfig && {
        SpotFleetRequestConfig: (0, models_4_1.SpotFleetRequestConfigDataFilterSensitiveLog)(obj.SpotFleetRequestConfig),
    }),
});
exports.RequestSpotFleetRequestFilterSensitiveLog = RequestSpotFleetRequestFilterSensitiveLog;
const RequestSpotFleetResponseFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RequestSpotFleetResponseFilterSensitiveLog = RequestSpotFleetResponseFilterSensitiveLog;
const RequestSpotLaunchSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.UserData && { UserData: smithy_client_1.SENSITIVE_STRING }),
});
exports.RequestSpotLaunchSpecificationFilterSensitiveLog = RequestSpotLaunchSpecificationFilterSensitiveLog;
const RequestSpotInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.LaunchSpecification && {
        LaunchSpecification: (0, exports.RequestSpotLaunchSpecificationFilterSensitiveLog)(obj.LaunchSpecification),
    }),
});
exports.RequestSpotInstancesRequestFilterSensitiveLog = RequestSpotInstancesRequestFilterSensitiveLog;
const RequestSpotInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.SpotInstanceRequests && {
        SpotInstanceRequests: obj.SpotInstanceRequests.map((item) => (0, models_4_1.SpotInstanceRequestFilterSensitiveLog)(item)),
    }),
});
exports.RequestSpotInstancesResultFilterSensitiveLog = RequestSpotInstancesResultFilterSensitiveLog;
const ResetAddressAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetAddressAttributeRequestFilterSensitiveLog = ResetAddressAttributeRequestFilterSensitiveLog;
const ResetAddressAttributeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetAddressAttributeResultFilterSensitiveLog = ResetAddressAttributeResultFilterSensitiveLog;
const ResetEbsDefaultKmsKeyIdRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetEbsDefaultKmsKeyIdRequestFilterSensitiveLog = ResetEbsDefaultKmsKeyIdRequestFilterSensitiveLog;
const ResetEbsDefaultKmsKeyIdResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetEbsDefaultKmsKeyIdResultFilterSensitiveLog = ResetEbsDefaultKmsKeyIdResultFilterSensitiveLog;
const ResetFpgaImageAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetFpgaImageAttributeRequestFilterSensitiveLog = ResetFpgaImageAttributeRequestFilterSensitiveLog;
const ResetFpgaImageAttributeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetFpgaImageAttributeResultFilterSensitiveLog = ResetFpgaImageAttributeResultFilterSensitiveLog;
const ResetImageAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetImageAttributeRequestFilterSensitiveLog = ResetImageAttributeRequestFilterSensitiveLog;
const ResetInstanceAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetInstanceAttributeRequestFilterSensitiveLog = ResetInstanceAttributeRequestFilterSensitiveLog;
const ResetNetworkInterfaceAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetNetworkInterfaceAttributeRequestFilterSensitiveLog = ResetNetworkInterfaceAttributeRequestFilterSensitiveLog;
const ResetSnapshotAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ResetSnapshotAttributeRequestFilterSensitiveLog = ResetSnapshotAttributeRequestFilterSensitiveLog;
const RestoreAddressToClassicRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreAddressToClassicRequestFilterSensitiveLog = RestoreAddressToClassicRequestFilterSensitiveLog;
const RestoreAddressToClassicResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreAddressToClassicResultFilterSensitiveLog = RestoreAddressToClassicResultFilterSensitiveLog;
const RestoreImageFromRecycleBinRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreImageFromRecycleBinRequestFilterSensitiveLog = RestoreImageFromRecycleBinRequestFilterSensitiveLog;
const RestoreImageFromRecycleBinResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreImageFromRecycleBinResultFilterSensitiveLog = RestoreImageFromRecycleBinResultFilterSensitiveLog;
const RestoreManagedPrefixListVersionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreManagedPrefixListVersionRequestFilterSensitiveLog = RestoreManagedPrefixListVersionRequestFilterSensitiveLog;
const RestoreManagedPrefixListVersionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreManagedPrefixListVersionResultFilterSensitiveLog = RestoreManagedPrefixListVersionResultFilterSensitiveLog;
const RestoreSnapshotFromRecycleBinRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreSnapshotFromRecycleBinRequestFilterSensitiveLog = RestoreSnapshotFromRecycleBinRequestFilterSensitiveLog;
const RestoreSnapshotFromRecycleBinResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreSnapshotFromRecycleBinResultFilterSensitiveLog = RestoreSnapshotFromRecycleBinResultFilterSensitiveLog;
const RestoreSnapshotTierRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreSnapshotTierRequestFilterSensitiveLog = RestoreSnapshotTierRequestFilterSensitiveLog;
const RestoreSnapshotTierResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RestoreSnapshotTierResultFilterSensitiveLog = RestoreSnapshotTierResultFilterSensitiveLog;
const RevokeClientVpnIngressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RevokeClientVpnIngressRequestFilterSensitiveLog = RevokeClientVpnIngressRequestFilterSensitiveLog;
const RevokeClientVpnIngressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RevokeClientVpnIngressResultFilterSensitiveLog = RevokeClientVpnIngressResultFilterSensitiveLog;
const RevokeSecurityGroupEgressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RevokeSecurityGroupEgressRequestFilterSensitiveLog = RevokeSecurityGroupEgressRequestFilterSensitiveLog;
const RevokeSecurityGroupEgressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RevokeSecurityGroupEgressResultFilterSensitiveLog = RevokeSecurityGroupEgressResultFilterSensitiveLog;
const RevokeSecurityGroupIngressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RevokeSecurityGroupIngressRequestFilterSensitiveLog = RevokeSecurityGroupIngressRequestFilterSensitiveLog;
const RevokeSecurityGroupIngressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RevokeSecurityGroupIngressResultFilterSensitiveLog = RevokeSecurityGroupIngressResultFilterSensitiveLog;
const CpuOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.CpuOptionsRequestFilterSensitiveLog = CpuOptionsRequestFilterSensitiveLog;
const ElasticInferenceAcceleratorFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ElasticInferenceAcceleratorFilterSensitiveLog = ElasticInferenceAcceleratorFilterSensitiveLog;
const EnclaveOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.EnclaveOptionsRequestFilterSensitiveLog = EnclaveOptionsRequestFilterSensitiveLog;
const HibernationOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.HibernationOptionsRequestFilterSensitiveLog = HibernationOptionsRequestFilterSensitiveLog;
const SpotMarketOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SpotMarketOptionsFilterSensitiveLog = SpotMarketOptionsFilterSensitiveLog;
const InstanceMarketOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceMarketOptionsRequestFilterSensitiveLog = InstanceMarketOptionsRequestFilterSensitiveLog;
const LaunchTemplateSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.LaunchTemplateSpecificationFilterSensitiveLog = LaunchTemplateSpecificationFilterSensitiveLog;
const LicenseConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.LicenseConfigurationRequestFilterSensitiveLog = LicenseConfigurationRequestFilterSensitiveLog;
const InstanceMaintenanceOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceMaintenanceOptionsRequestFilterSensitiveLog = InstanceMaintenanceOptionsRequestFilterSensitiveLog;
const InstanceMetadataOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceMetadataOptionsRequestFilterSensitiveLog = InstanceMetadataOptionsRequestFilterSensitiveLog;
const PrivateDnsNameOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.PrivateDnsNameOptionsRequestFilterSensitiveLog = PrivateDnsNameOptionsRequestFilterSensitiveLog;
const RunInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.UserData && { UserData: smithy_client_1.SENSITIVE_STRING }),
});
exports.RunInstancesRequestFilterSensitiveLog = RunInstancesRequestFilterSensitiveLog;
const ScheduledInstancesEbsFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ScheduledInstancesEbsFilterSensitiveLog = ScheduledInstancesEbsFilterSensitiveLog;
const ScheduledInstancesBlockDeviceMappingFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ScheduledInstancesBlockDeviceMappingFilterSensitiveLog = ScheduledInstancesBlockDeviceMappingFilterSensitiveLog;
const ScheduledInstancesIamInstanceProfileFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ScheduledInstancesIamInstanceProfileFilterSensitiveLog = ScheduledInstancesIamInstanceProfileFilterSensitiveLog;
const ScheduledInstancesMonitoringFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ScheduledInstancesMonitoringFilterSensitiveLog = ScheduledInstancesMonitoringFilterSensitiveLog;
const ScheduledInstancesIpv6AddressFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ScheduledInstancesIpv6AddressFilterSensitiveLog = ScheduledInstancesIpv6AddressFilterSensitiveLog;
const ScheduledInstancesPrivateIpAddressConfigFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ScheduledInstancesPrivateIpAddressConfigFilterSensitiveLog = ScheduledInstancesPrivateIpAddressConfigFilterSensitiveLog;
const ScheduledInstancesNetworkInterfaceFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ScheduledInstancesNetworkInterfaceFilterSensitiveLog = ScheduledInstancesNetworkInterfaceFilterSensitiveLog;
const ScheduledInstancesPlacementFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ScheduledInstancesPlacementFilterSensitiveLog = ScheduledInstancesPlacementFilterSensitiveLog;
const ScheduledInstancesLaunchSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.ScheduledInstancesLaunchSpecificationFilterSensitiveLog = ScheduledInstancesLaunchSpecificationFilterSensitiveLog;
const RunScheduledInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.LaunchSpecification && { LaunchSpecification: smithy_client_1.SENSITIVE_STRING }),
});
exports.RunScheduledInstancesRequestFilterSensitiveLog = RunScheduledInstancesRequestFilterSensitiveLog;
const RunScheduledInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.RunScheduledInstancesResultFilterSensitiveLog = RunScheduledInstancesResultFilterSensitiveLog;
const SearchLocalGatewayRoutesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SearchLocalGatewayRoutesRequestFilterSensitiveLog = SearchLocalGatewayRoutesRequestFilterSensitiveLog;
const SearchLocalGatewayRoutesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SearchLocalGatewayRoutesResultFilterSensitiveLog = SearchLocalGatewayRoutesResultFilterSensitiveLog;
const SearchTransitGatewayMulticastGroupsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SearchTransitGatewayMulticastGroupsRequestFilterSensitiveLog = SearchTransitGatewayMulticastGroupsRequestFilterSensitiveLog;
const TransitGatewayMulticastGroupFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TransitGatewayMulticastGroupFilterSensitiveLog = TransitGatewayMulticastGroupFilterSensitiveLog;
const SearchTransitGatewayMulticastGroupsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SearchTransitGatewayMulticastGroupsResultFilterSensitiveLog = SearchTransitGatewayMulticastGroupsResultFilterSensitiveLog;
const SearchTransitGatewayRoutesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SearchTransitGatewayRoutesRequestFilterSensitiveLog = SearchTransitGatewayRoutesRequestFilterSensitiveLog;
const SearchTransitGatewayRoutesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SearchTransitGatewayRoutesResultFilterSensitiveLog = SearchTransitGatewayRoutesResultFilterSensitiveLog;
const SendDiagnosticInterruptRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.SendDiagnosticInterruptRequestFilterSensitiveLog = SendDiagnosticInterruptRequestFilterSensitiveLog;
const StartInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StartInstancesRequestFilterSensitiveLog = StartInstancesRequestFilterSensitiveLog;
const InstanceStateChangeFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.InstanceStateChangeFilterSensitiveLog = InstanceStateChangeFilterSensitiveLog;
const StartInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StartInstancesResultFilterSensitiveLog = StartInstancesResultFilterSensitiveLog;
const StartNetworkInsightsAccessScopeAnalysisRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StartNetworkInsightsAccessScopeAnalysisRequestFilterSensitiveLog = StartNetworkInsightsAccessScopeAnalysisRequestFilterSensitiveLog;
const StartNetworkInsightsAccessScopeAnalysisResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StartNetworkInsightsAccessScopeAnalysisResultFilterSensitiveLog = StartNetworkInsightsAccessScopeAnalysisResultFilterSensitiveLog;
const StartNetworkInsightsAnalysisRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StartNetworkInsightsAnalysisRequestFilterSensitiveLog = StartNetworkInsightsAnalysisRequestFilterSensitiveLog;
const StartNetworkInsightsAnalysisResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StartNetworkInsightsAnalysisResultFilterSensitiveLog = StartNetworkInsightsAnalysisResultFilterSensitiveLog;
const StartVpcEndpointServicePrivateDnsVerificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StartVpcEndpointServicePrivateDnsVerificationRequestFilterSensitiveLog = StartVpcEndpointServicePrivateDnsVerificationRequestFilterSensitiveLog;
const StartVpcEndpointServicePrivateDnsVerificationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StartVpcEndpointServicePrivateDnsVerificationResultFilterSensitiveLog = StartVpcEndpointServicePrivateDnsVerificationResultFilterSensitiveLog;
const StopInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StopInstancesRequestFilterSensitiveLog = StopInstancesRequestFilterSensitiveLog;
const StopInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.StopInstancesResultFilterSensitiveLog = StopInstancesResultFilterSensitiveLog;
const TerminateClientVpnConnectionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TerminateClientVpnConnectionsRequestFilterSensitiveLog = TerminateClientVpnConnectionsRequestFilterSensitiveLog;
const TerminateConnectionStatusFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TerminateConnectionStatusFilterSensitiveLog = TerminateConnectionStatusFilterSensitiveLog;
const TerminateClientVpnConnectionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TerminateClientVpnConnectionsResultFilterSensitiveLog = TerminateClientVpnConnectionsResultFilterSensitiveLog;
const TerminateInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TerminateInstancesRequestFilterSensitiveLog = TerminateInstancesRequestFilterSensitiveLog;
const TerminateInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
exports.TerminateInstancesResultFilterSensitiveLog = TerminateInstancesResultFilterSensitiveLog;
