// @aws-sdk/client-ec2/dist-es/models/models_6.js
import { SENSITIVE_STRING } from "@aws-sdk/smithy-client";
import { SpotFleetRequestConfigDataFilterSensitiveLog, SpotInstanceRequestFilterSensitiveLog, } from "./models_4";
export var ModifyAvailabilityZoneOptInStatus;
(function (ModifyAvailabilityZoneOptInStatus) {
    ModifyAvailabilityZoneOptInStatus["not_opted_in"] = "not-opted-in";
    ModifyAvailabilityZoneOptInStatus["opted_in"] = "opted-in";
})(ModifyAvailabilityZoneOptInStatus || (ModifyAvailabilityZoneOptInStatus = {}));
export var OperationType;
(function (OperationType) {
    OperationType["add"] = "add";
    OperationType["remove"] = "remove";
})(OperationType || (OperationType = {}));
export var UnsuccessfulInstanceCreditSpecificationErrorCode;
(function (UnsuccessfulInstanceCreditSpecificationErrorCode) {
    UnsuccessfulInstanceCreditSpecificationErrorCode["INCORRECT_INSTANCE_STATE"] = "IncorrectInstanceState";
    UnsuccessfulInstanceCreditSpecificationErrorCode["INSTANCE_CREDIT_SPECIFICATION_NOT_SUPPORTED"] = "InstanceCreditSpecification.NotSupported";
    UnsuccessfulInstanceCreditSpecificationErrorCode["INSTANCE_NOT_FOUND"] = "InvalidInstanceID.NotFound";
    UnsuccessfulInstanceCreditSpecificationErrorCode["INVALID_INSTANCE_ID"] = "InvalidInstanceID.Malformed";
})(UnsuccessfulInstanceCreditSpecificationErrorCode || (UnsuccessfulInstanceCreditSpecificationErrorCode = {}));
export var HostTenancy;
(function (HostTenancy) {
    HostTenancy["dedicated"] = "dedicated";
    HostTenancy["host"] = "host";
})(HostTenancy || (HostTenancy = {}));
export var TargetStorageTier;
(function (TargetStorageTier) {
    TargetStorageTier["archive"] = "archive";
})(TargetStorageTier || (TargetStorageTier = {}));
export var TrafficMirrorFilterRuleField;
(function (TrafficMirrorFilterRuleField) {
    TrafficMirrorFilterRuleField["description"] = "description";
    TrafficMirrorFilterRuleField["destination_port_range"] = "destination-port-range";
    TrafficMirrorFilterRuleField["protocol"] = "protocol";
    TrafficMirrorFilterRuleField["source_port_range"] = "source-port-range";
})(TrafficMirrorFilterRuleField || (TrafficMirrorFilterRuleField = {}));
export var TrafficMirrorSessionField;
(function (TrafficMirrorSessionField) {
    TrafficMirrorSessionField["description"] = "description";
    TrafficMirrorSessionField["packet_length"] = "packet-length";
    TrafficMirrorSessionField["virtual_network_id"] = "virtual-network-id";
})(TrafficMirrorSessionField || (TrafficMirrorSessionField = {}));
export var VpcTenancy;
(function (VpcTenancy) {
    VpcTenancy["default"] = "default";
})(VpcTenancy || (VpcTenancy = {}));
export var Status;
(function (Status) {
    Status["inClassic"] = "InClassic";
    Status["inVpc"] = "InVpc";
    Status["moveInProgress"] = "MoveInProgress";
})(Status || (Status = {}));
export var ReportInstanceReasonCodes;
(function (ReportInstanceReasonCodes) {
    ReportInstanceReasonCodes["instance_stuck_in_state"] = "instance-stuck-in-state";
    ReportInstanceReasonCodes["not_accepting_credentials"] = "not-accepting-credentials";
    ReportInstanceReasonCodes["other"] = "other";
    ReportInstanceReasonCodes["password_not_available"] = "password-not-available";
    ReportInstanceReasonCodes["performance_ebs_volume"] = "performance-ebs-volume";
    ReportInstanceReasonCodes["performance_instance_store"] = "performance-instance-store";
    ReportInstanceReasonCodes["performance_network"] = "performance-network";
    ReportInstanceReasonCodes["performance_other"] = "performance-other";
    ReportInstanceReasonCodes["unresponsive"] = "unresponsive";
})(ReportInstanceReasonCodes || (ReportInstanceReasonCodes = {}));
export var ReportStatusType;
(function (ReportStatusType) {
    ReportStatusType["impaired"] = "impaired";
    ReportStatusType["ok"] = "ok";
})(ReportStatusType || (ReportStatusType = {}));
export var ResetFpgaImageAttributeName;
(function (ResetFpgaImageAttributeName) {
    ResetFpgaImageAttributeName["loadPermission"] = "loadPermission";
})(ResetFpgaImageAttributeName || (ResetFpgaImageAttributeName = {}));
export var ResetImageAttributeName;
(function (ResetImageAttributeName) {
    ResetImageAttributeName["launchPermission"] = "launchPermission";
})(ResetImageAttributeName || (ResetImageAttributeName = {}));
export var MembershipType;
(function (MembershipType) {
    MembershipType["igmp"] = "igmp";
    MembershipType["static"] = "static";
})(MembershipType || (MembershipType = {}));
export const ModifyAvailabilityZoneGroupRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyAvailabilityZoneGroupResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyCapacityReservationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyCapacityReservationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyCapacityReservationFleetRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyCapacityReservationFleetResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const DnsServersOptionsModifyStructureFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyClientVpnEndpointRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyClientVpnEndpointResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyDefaultCreditSpecificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyDefaultCreditSpecificationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyEbsDefaultKmsKeyIdRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyEbsDefaultKmsKeyIdResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyFleetRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyFleetResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const LoadPermissionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const LoadPermissionModificationsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyFpgaImageAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyFpgaImageAttributeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyHostsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyHostsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIdentityIdFormatRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIdFormatRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const LaunchPermissionModificationsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyImageAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const EbsInstanceBlockDeviceSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const InstanceBlockDeviceMappingSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const BlobAttributeValueFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const CapacityReservationSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceCapacityReservationAttributesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceCapacityReservationAttributesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const InstanceCreditSpecificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceCreditSpecificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SuccessfulInstanceCreditSpecificationItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UnsuccessfulInstanceCreditSpecificationItemErrorFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UnsuccessfulInstanceCreditSpecificationItemFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceCreditSpecificationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceEventStartTimeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceEventStartTimeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceEventWindowRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceEventWindowResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceMaintenanceOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceMaintenanceOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceMetadataOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstanceMetadataOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstancePlacementRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyInstancePlacementResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RemoveIpamOperatingRegionFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamPoolRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamPoolResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamResourceCidrRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamResourceCidrResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamResourceDiscoveryRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamResourceDiscoveryResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamScopeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyIpamScopeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyLaunchTemplateRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyLaunchTemplateResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyLocalGatewayRouteRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyLocalGatewayRouteResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RemovePrefixListEntryFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyManagedPrefixListRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyManagedPrefixListResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const NetworkInterfaceAttachmentChangesFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyNetworkInterfaceAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyPrivateDnsNameOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyPrivateDnsNameOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyReservedInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyReservedInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SecurityGroupRuleRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SecurityGroupRuleUpdateFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifySecurityGroupRulesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifySecurityGroupRulesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const CreateVolumePermissionModificationsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifySnapshotAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifySnapshotTierRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifySnapshotTierResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifySpotFleetRequestRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifySpotFleetRequestResponseFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifySubnetAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTrafficMirrorFilterNetworkServicesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTrafficMirrorFilterNetworkServicesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTrafficMirrorFilterRuleRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTrafficMirrorFilterRuleResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTrafficMirrorSessionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTrafficMirrorSessionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTransitGatewayOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTransitGatewayRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTransitGatewayResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTransitGatewayPrefixListReferenceRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTransitGatewayPrefixListReferenceResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTransitGatewayVpcAttachmentRequestOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTransitGatewayVpcAttachmentRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyTransitGatewayVpcAttachmentResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessEndpointLoadBalancerOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessEndpointEniOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessEndpointRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessEndpointResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessEndpointPolicyRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessEndpointPolicyResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessGroupRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessGroupResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessGroupPolicyRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessGroupPolicyResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessInstanceRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessInstanceResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const VerifiedAccessLogCloudWatchLogsDestinationOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const VerifiedAccessLogKinesisDataFirehoseDestinationOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const VerifiedAccessLogS3DestinationOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const VerifiedAccessLogOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessInstanceLoggingConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessInstanceLoggingConfigurationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessTrustProviderOidcOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessTrustProviderRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVerifiedAccessTrustProviderResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVolumeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVolumeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVolumeAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointConnectionNotificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointConnectionNotificationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointServiceConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointServiceConfigurationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointServicePayerResponsibilityRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointServicePayerResponsibilityResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointServicePermissionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcEndpointServicePermissionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PeeringConnectionOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcPeeringConnectionOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PeeringConnectionOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcPeeringConnectionOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcTenancyRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpcTenancyResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpnConnectionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpnConnectionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpnConnectionOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpnConnectionOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpnTunnelCertificateRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpnTunnelCertificateResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpnTunnelOptionsSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpnTunnelOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ModifyVpnTunnelOptionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const MonitorInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const InstanceMonitoringFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const MonitorInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const MoveAddressToVpcRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const MoveAddressToVpcResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const MoveByoipCidrToIpamRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const MoveByoipCidrToIpamResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const CidrAuthorizationContextFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ProvisionByoipCidrRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ProvisionByoipCidrResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const IpamCidrAuthorizationContextFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ProvisionIpamPoolCidrRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ProvisionIpamPoolCidrResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ProvisionPublicIpv4PoolCidrRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ProvisionPublicIpv4PoolCidrResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PurchaseHostReservationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PurchaseHostReservationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReservedInstanceLimitPriceFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PurchaseReservedInstancesOfferingRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PurchaseReservedInstancesOfferingResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PurchaseRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PurchaseScheduledInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PurchaseScheduledInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RebootInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RegisterImageRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RegisterImageResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RegisterInstanceTagAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RegisterInstanceEventNotificationAttributesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RegisterInstanceEventNotificationAttributesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RegisterTransitGatewayMulticastGroupMembersRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const TransitGatewayMulticastRegisteredGroupMembersFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RegisterTransitGatewayMulticastGroupMembersResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RegisterTransitGatewayMulticastGroupSourcesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const TransitGatewayMulticastRegisteredGroupSourcesFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RegisterTransitGatewayMulticastGroupSourcesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectTransitGatewayMulticastDomainAssociationsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectTransitGatewayMulticastDomainAssociationsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectTransitGatewayPeeringAttachmentRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectTransitGatewayPeeringAttachmentResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectTransitGatewayVpcAttachmentRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectTransitGatewayVpcAttachmentResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectVpcEndpointConnectionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectVpcEndpointConnectionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectVpcPeeringConnectionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RejectVpcPeeringConnectionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReleaseAddressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReleaseHostsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReleaseHostsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReleaseIpamPoolAllocationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReleaseIpamPoolAllocationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceIamInstanceProfileAssociationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceIamInstanceProfileAssociationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceNetworkAclAssociationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceNetworkAclAssociationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceNetworkAclEntryRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceRouteRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceRouteTableAssociationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceRouteTableAssociationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceTransitGatewayRouteRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReplaceTransitGatewayRouteResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ReportInstanceStatusRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RequestSpotFleetRequestFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.SpotFleetRequestConfig && {
        SpotFleetRequestConfig: SpotFleetRequestConfigDataFilterSensitiveLog(obj.SpotFleetRequestConfig),
    }),
});
export const RequestSpotFleetResponseFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RequestSpotLaunchSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.UserData && { UserData: SENSITIVE_STRING }),
});
export const RequestSpotInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.LaunchSpecification && {
        LaunchSpecification: RequestSpotLaunchSpecificationFilterSensitiveLog(obj.LaunchSpecification),
    }),
});
export const RequestSpotInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.SpotInstanceRequests && {
        SpotInstanceRequests: obj.SpotInstanceRequests.map((item) => SpotInstanceRequestFilterSensitiveLog(item)),
    }),
});
export const ResetAddressAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ResetAddressAttributeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ResetEbsDefaultKmsKeyIdRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ResetEbsDefaultKmsKeyIdResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ResetFpgaImageAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ResetFpgaImageAttributeResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ResetImageAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ResetInstanceAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ResetNetworkInterfaceAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ResetSnapshotAttributeRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreAddressToClassicRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreAddressToClassicResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreImageFromRecycleBinRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreImageFromRecycleBinResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreManagedPrefixListVersionRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreManagedPrefixListVersionResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreSnapshotFromRecycleBinRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreSnapshotFromRecycleBinResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreSnapshotTierRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RestoreSnapshotTierResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RevokeClientVpnIngressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RevokeClientVpnIngressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RevokeSecurityGroupEgressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RevokeSecurityGroupEgressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RevokeSecurityGroupIngressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RevokeSecurityGroupIngressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const CpuOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ElasticInferenceAcceleratorFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const EnclaveOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const HibernationOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SpotMarketOptionsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const InstanceMarketOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const LaunchTemplateSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const LicenseConfigurationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const InstanceMaintenanceOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const InstanceMetadataOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const PrivateDnsNameOptionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RunInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.UserData && { UserData: SENSITIVE_STRING }),
});
export const ScheduledInstancesEbsFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ScheduledInstancesBlockDeviceMappingFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ScheduledInstancesIamInstanceProfileFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ScheduledInstancesMonitoringFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ScheduledInstancesIpv6AddressFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ScheduledInstancesPrivateIpAddressConfigFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ScheduledInstancesNetworkInterfaceFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ScheduledInstancesPlacementFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const ScheduledInstancesLaunchSpecificationFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const RunScheduledInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
    ...(obj.LaunchSpecification && { LaunchSpecification: SENSITIVE_STRING }),
});
export const RunScheduledInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SearchLocalGatewayRoutesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SearchLocalGatewayRoutesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SearchTransitGatewayMulticastGroupsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const TransitGatewayMulticastGroupFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SearchTransitGatewayMulticastGroupsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SearchTransitGatewayRoutesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SearchTransitGatewayRoutesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SendDiagnosticInterruptRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StartInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const InstanceStateChangeFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StartInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StartNetworkInsightsAccessScopeAnalysisRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StartNetworkInsightsAccessScopeAnalysisResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StartNetworkInsightsAnalysisRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StartNetworkInsightsAnalysisResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StartVpcEndpointServicePrivateDnsVerificationRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StartVpcEndpointServicePrivateDnsVerificationResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StopInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const StopInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const TerminateClientVpnConnectionsRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const TerminateConnectionStatusFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const TerminateClientVpnConnectionsResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const TerminateInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const TerminateInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
