// @aws-sdk/client-ec2/dist-es/models/models_7.js
export const UnassignIpv6AddressesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UnassignIpv6AddressesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UnassignPrivateIpAddressesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UnassignPrivateNatGatewayAddressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UnassignPrivateNatGatewayAddressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UnmonitorInstancesRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UnmonitorInstancesResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const SecurityGroupRuleDescriptionFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UpdateSecurityGroupRuleDescriptionsEgressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UpdateSecurityGroupRuleDescriptionsEgressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UpdateSecurityGroupRuleDescriptionsIngressRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const UpdateSecurityGroupRuleDescriptionsIngressResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const WithdrawByoipCidrRequestFilterSensitiveLog = (obj) => ({
    ...obj,
});
export const WithdrawByoipCidrResultFilterSensitiveLog = (obj) => ({
    ...obj,
});
