// @aws-sdk/client-ec2/dist-es/commands/CreateNetworkInterfacePermissionCommand.js
import { getEndpointPlugin } from "@aws-sdk/middleware-endpoint";
import { getSerdePlugin } from "@aws-sdk/middleware-serde";
import { Command as $Command } from "@aws-sdk/smithy-client";
import { CreateNetworkInterfacePermissionRequestFilterSensitiveLog, CreateNetworkInterfacePermissionResultFilterSensitiveLog, } from "../models/models_1";
import { deserializeAws_ec2CreateNetworkInterfacePermissionCommand, serializeAws_ec2CreateNetworkInterfacePermissionCommand, } from "../protocols/Aws_ec2";
export class CreateNetworkInterfacePermissionCommand extends $Command {
    constructor(input) {
        super();
        this.input = input;
    }
    static getEndpointParameterInstructions() {
        return {
            UseFIPS: { type: "builtInParams", name: "useFipsEndpoint" },
            Endpoint: { type: "builtInParams", name: "endpoint" },
            Region: { type: "builtInParams", name: "region" },
            UseDualStack: { type: "builtInParams", name: "useDualstackEndpoint" },
        };
    }
    resolveMiddleware(clientStack, configuration, options) {
        this.middlewareStack.use(getSerdePlugin(configuration, this.serialize, this.deserialize));
        this.middlewareStack.use(getEndpointPlugin(configuration, CreateNetworkInterfacePermissionCommand.getEndpointParameterInstructions()));
        const stack = clientStack.concat(this.middlewareStack);
        const { logger } = configuration;
        const clientName = "EC2Client";
        const commandName = "CreateNetworkInterfacePermissionCommand";
        const handlerExecutionContext = {
            logger,
            clientName,
            commandName,
            inputFilterSensitiveLog: CreateNetworkInterfacePermissionRequestFilterSensitiveLog,
            outputFilterSensitiveLog: CreateNetworkInterfacePermissionResultFilterSensitiveLog,
        };
        const { requestHandler } = configuration;
        return stack.resolve((request) => requestHandler.handle(request.request, options || {}), handlerExecutionContext);
    }
    serialize(input, context) {
        return serializeAws_ec2CreateNetworkInterfacePermissionCommand(input, context);
    }
    deserialize(output, context) {
        return deserializeAws_ec2CreateNetworkInterfacePermissionCommand(output, context);
    }
}
