// @aws-sdk/client-ec2/dist-es/commands/ReplaceNetworkAclEntryCommand.js
import { getEndpointPlugin } from "@aws-sdk/middleware-endpoint";
import { getSerdePlugin } from "@aws-sdk/middleware-serde";
import { Command as $Command } from "@aws-sdk/smithy-client";
import { ReplaceNetworkAclEntryRequestFilterSensitiveLog } from "../models/models_6";
import { deserializeAws_ec2ReplaceNetworkAclEntryCommand, serializeAws_ec2ReplaceNetworkAclEntryCommand, } from "../protocols/Aws_ec2";
export class ReplaceNetworkAclEntryCommand extends $Command {
    constructor(input) {
        super();
        this.input = input;
    }
    static getEndpointParameterInstructions() {
        return {
            UseFIPS: { type: "builtInParams", name: "useFipsEndpoint" },
            Endpoint: { type: "builtInParams", name: "endpoint" },
            Region: { type: "builtInParams", name: "region" },
            UseDualStack: { type: "builtInParams", name: "useDualstackEndpoint" },
        };
    }
    resolveMiddleware(clientStack, configuration, options) {
        this.middlewareStack.use(getSerdePlugin(configuration, this.serialize, this.deserialize));
        this.middlewareStack.use(getEndpointPlugin(configuration, ReplaceNetworkAclEntryCommand.getEndpointParameterInstructions()));
        const stack = clientStack.concat(this.middlewareStack);
        const { logger } = configuration;
        const clientName = "EC2Client";
        const commandName = "ReplaceNetworkAclEntryCommand";
        const handlerExecutionContext = {
            logger,
            clientName,
            commandName,
            inputFilterSensitiveLog: ReplaceNetworkAclEntryRequestFilterSensitiveLog,
            outputFilterSensitiveLog: (output) => output,
        };
        const { requestHandler } = configuration;
        return stack.resolve((request) => requestHandler.handle(request.request, options || {}), handlerExecutionContext);
    }
    serialize(input, context) {
        return serializeAws_ec2ReplaceNetworkAclEntryCommand(input, context);
    }
    deserialize(output, context) {
        return deserializeAws_ec2ReplaceNetworkAclEntryCommand(output, context);
    }
}
